<!DOCTYPE html><!--[if lt IE 7]> <html class="ie6 ie"> <![endif]--><!--[if IE 7]>    <html class="ie7 ie"> <![endif]--><!--[if IE 8]>    <html class="ie8 ie"> <![endif]--><!--[if IE 9]>    <html class="ie9 ie"> <![endif]--><!--[if !IE]> --><html xmlns="http://www.w3.org/1999/xhtml"><!-- <![endif]--><head>
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<title>
Replication and Reproduction (Stanford Encyclopedia of Philosophy/Winter 2019 Edition)
</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="robots" content="noarchive, noodp" />
<meta property="citation_title" content="Replication and Reproduction" />
<meta property="citation_author" content="Wilkins, John S." />
<meta property="citation_author" content="Bourrat, Pierrick" />
<meta property="citation_publication_date" content="2001/12/05" />
<meta name="DC.title" content="Replication and Reproduction" />
<meta name="DC.creator" content="Wilkins, John S." />
<meta name="DC.creator" content="Bourrat, Pierrick" />
<meta name="DCTERMS.issued" content="2001-12-05" />
<meta name="DCTERMS.modified" content="2018-09-25" />

<!-- NOTE: Import webfonts using this link: -->
<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,300,600,200&amp;subset=latin,latin-ext" rel="stylesheet" type="text/css" />

<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap.min.css" />
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap-responsive.min.css" />
<link rel="stylesheet" type="text/css" href="../../css/font-awesome.min.css" />
<!--[if IE 7]> <link rel="stylesheet" type="text/css" href="../../css/font-awesome-ie7.min.css"> <![endif]-->
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/style.css" />
<link rel="stylesheet" type="text/css" media="print" href="../../css/print.css" />
<link rel="stylesheet" type="text/css" href="../../css/entry.css" />
<!--[if IE]> <link rel="stylesheet" type="text/css" href="../../css/ie.css" /> <![endif]-->
<script type="text/javascript" src="../../js/jquery-1.9.1.min.js"></script>
<script type="text/javascript" src="../../js/bootstrap.min.js"></script>

<!-- NOTE: Javascript for sticky behavior needed on article and ToC pages -->
<script type="text/javascript" src="../../js/jquery-scrolltofixed-min.js"></script>
<script type="text/javascript" src="../../js/entry.js"></script>

<!-- SEP custom script -->
<script type="text/javascript" src="../../js/sep.js"></script>
<style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom &gt; * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd &gt; * {display: block!important}
.MJXp-script-box &gt; *  {display: table!important; height: 50%}
.MJXp-script-box &gt; * &gt; * {display: table-cell!important; vertical-align: top}
.MJXp-script-box &gt; *:last-child &gt; * {vertical-align: bottom}
.MJXp-script-box &gt; * &gt; * &gt; * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over &gt; * {display: block!important}
.MJXp-munderover &gt; * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable &gt; * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr &gt; .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child &gt; .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr &gt; .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child &gt; .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><style type="text/css">.mjx-chtml {display: inline-block; line-height: 0; text-indent: 0; text-align: left; text-transform: none; font-style: normal; font-weight: normal; font-size: 100%; font-size-adjust: none; letter-spacing: normal; word-wrap: normal; word-spacing: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; margin: 0; padding: 1px 0}
.MJXc-display {display: block; text-align: center; margin: 1em 0; padding: 0}
.mjx-chtml[tabindex]:focus, body :focus .mjx-chtml[tabindex] {display: inline-table}
.mjx-full-width {text-align: center; display: table-cell!important; width: 10000em}
.mjx-math {display: inline-block; border-collapse: separate; border-spacing: 0}
.mjx-math * {display: inline-block; -webkit-box-sizing: content-box!important; -moz-box-sizing: content-box!important; box-sizing: content-box!important; text-align: left}
.mjx-numerator {display: block; text-align: center}
.mjx-denominator {display: block; text-align: center}
.MJXc-stacked {height: 0; position: relative}
.MJXc-stacked &gt; * {position: absolute}
.MJXc-bevelled &gt; * {display: inline-block}
.mjx-stack {display: inline-block}
.mjx-op {display: block}
.mjx-under {display: table-cell}
.mjx-over {display: block}
.mjx-over &gt; * {padding-left: 0px!important; padding-right: 0px!important}
.mjx-under &gt; * {padding-left: 0px!important; padding-right: 0px!important}
.mjx-stack &gt; .mjx-sup {display: block}
.mjx-stack &gt; .mjx-sub {display: block}
.mjx-prestack &gt; .mjx-presup {display: block}
.mjx-prestack &gt; .mjx-presub {display: block}
.mjx-delim-h &gt; .mjx-char {display: inline-block}
.mjx-surd {vertical-align: top}
.mjx-mphantom * {visibility: hidden}
.mjx-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 2px 3px; font-style: normal; font-size: 90%}
.mjx-annotation-xml {line-height: normal}
.mjx-menclose &gt; svg {fill: none; stroke: currentColor}
.mjx-mtr {display: table-row}
.mjx-mlabeledtr {display: table-row}
.mjx-mtd {display: table-cell; text-align: center}
.mjx-label {display: table-row}
.mjx-box {display: inline-block}
.mjx-block {display: block}
.mjx-span {display: inline}
.mjx-char {display: block; white-space: pre}
.mjx-itable {display: inline-table; width: auto}
.mjx-row {display: table-row}
.mjx-cell {display: table-cell}
.mjx-table {display: table; width: 100%}
.mjx-line {display: block; height: 0}
.mjx-strut {width: 0; padding-top: 1em}
.mjx-vsize {width: 0}
.MJXc-space1 {margin-left: .167em}
.MJXc-space2 {margin-left: .222em}
.MJXc-space3 {margin-left: .278em}
.mjx-chartest {display: block; visibility: hidden; position: absolute; top: 0; line-height: normal; font-size: 500%}
.mjx-chartest .mjx-char {display: inline}
.mjx-chartest .mjx-box {padding-top: 1000px}
.MJXc-processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MJXc-processed {display: none}
.mjx-test {display: block; font-style: normal; font-weight: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-transform: none; letter-spacing: normal; word-spacing: normal; overflow: hidden; height: 1px}
.mjx-ex-box-test {position: absolute; overflow: hidden; width: 1px; height: 60ex}
.mjx-line-box-test {display: table!important}
.mjx-line-box-test span {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
#MathJax_CHTML_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.mjx-chtml .mjx-noError {line-height: 1.2; vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
.MJXc-TeX-unknown-R {font-family: STIXGeneral,'Cambria Math','Arial Unicode MS',serif; font-style: normal; font-weight: normal}
.MJXc-TeX-unknown-I {font-family: STIXGeneral,'Cambria Math','Arial Unicode MS',serif; font-style: italic; font-weight: normal}
.MJXc-TeX-unknown-B {font-family: STIXGeneral,'Cambria Math','Arial Unicode MS',serif; font-style: normal; font-weight: bold}
.MJXc-TeX-unknown-BI {font-family: STIXGeneral,'Cambria Math','Arial Unicode MS',serif; font-style: italic; font-weight: bold}
.MJXc-TeX-ams-R {font-family: MJXc-TeX-ams-R,MJXc-TeX-ams-Rw}
.MJXc-TeX-cal-B {font-family: MJXc-TeX-cal-B,MJXc-TeX-cal-Bx,MJXc-TeX-cal-Bw}
.MJXc-TeX-frak-R {font-family: MJXc-TeX-frak-R,MJXc-TeX-frak-Rw}
.MJXc-TeX-frak-B {font-family: MJXc-TeX-frak-B,MJXc-TeX-frak-Bx,MJXc-TeX-frak-Bw}
.MJXc-TeX-math-BI {font-family: MJXc-TeX-math-BI,MJXc-TeX-math-BIx,MJXc-TeX-math-BIw}
.MJXc-TeX-sans-R {font-family: MJXc-TeX-sans-R,MJXc-TeX-sans-Rw}
.MJXc-TeX-sans-B {font-family: MJXc-TeX-sans-B,MJXc-TeX-sans-Bx,MJXc-TeX-sans-Bw}
.MJXc-TeX-sans-I {font-family: MJXc-TeX-sans-I,MJXc-TeX-sans-Ix,MJXc-TeX-sans-Iw}
.MJXc-TeX-script-R {font-family: MJXc-TeX-script-R,MJXc-TeX-script-Rw}
.MJXc-TeX-type-R {font-family: MJXc-TeX-type-R,MJXc-TeX-type-Rw}
.MJXc-TeX-cal-R {font-family: MJXc-TeX-cal-R,MJXc-TeX-cal-Rw}
.MJXc-TeX-main-B {font-family: MJXc-TeX-main-B,MJXc-TeX-main-Bx,MJXc-TeX-main-Bw}
.MJXc-TeX-main-I {font-family: MJXc-TeX-main-I,MJXc-TeX-main-Ix,MJXc-TeX-main-Iw}
.MJXc-TeX-main-R {font-family: MJXc-TeX-main-R,MJXc-TeX-main-Rw}
.MJXc-TeX-math-I {font-family: MJXc-TeX-math-I,MJXc-TeX-math-Ix,MJXc-TeX-math-Iw}
.MJXc-TeX-size1-R {font-family: MJXc-TeX-size1-R,MJXc-TeX-size1-Rw}
.MJXc-TeX-size2-R {font-family: MJXc-TeX-size2-R,MJXc-TeX-size2-Rw}
.MJXc-TeX-size3-R {font-family: MJXc-TeX-size3-R,MJXc-TeX-size3-Rw}
.MJXc-TeX-size4-R {font-family: MJXc-TeX-size4-R,MJXc-TeX-size4-Rw}
.MJXc-TeX-vec-R {font-family: MJXc-TeX-vec-R,MJXc-TeX-vec-Rw}
.MJXc-TeX-vec-B {font-family: MJXc-TeX-vec-B,MJXc-TeX-vec-Bx,MJXc-TeX-vec-Bw}
@font-face {font-family: MJXc-TeX-ams-R; src: local('MathJax_AMS'), local('MathJax_AMS-Regular')}
@font-face {font-family: MJXc-TeX-ams-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_AMS-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_AMS-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_AMS-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-cal-B; src: local('MathJax_Caligraphic Bold'), local('MathJax_Caligraphic-Bold')}
@font-face {font-family: MJXc-TeX-cal-Bx; src: local('MathJax_Caligraphic'); font-weight: bold}
@font-face {font-family: MJXc-TeX-cal-Bw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Caligraphic-Bold.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Bold.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Bold.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-frak-R; src: local('MathJax_Fraktur'), local('MathJax_Fraktur-Regular')}
@font-face {font-family: MJXc-TeX-frak-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Fraktur-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Fraktur-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Fraktur-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-frak-B; src: local('MathJax_Fraktur Bold'), local('MathJax_Fraktur-Bold')}
@font-face {font-family: MJXc-TeX-frak-Bx; src: local('MathJax_Fraktur'); font-weight: bold}
@font-face {font-family: MJXc-TeX-frak-Bw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Fraktur-Bold.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Fraktur-Bold.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Fraktur-Bold.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-math-BI; src: local('MathJax_Math BoldItalic'), local('MathJax_Math-BoldItalic')}
@font-face {font-family: MJXc-TeX-math-BIx; src: local('MathJax_Math'); font-weight: bold; font-style: italic}
@font-face {font-family: MJXc-TeX-math-BIw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Math-BoldItalic.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Math-BoldItalic.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Math-BoldItalic.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-sans-R; src: local('MathJax_SansSerif'), local('MathJax_SansSerif-Regular')}
@font-face {font-family: MJXc-TeX-sans-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_SansSerif-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_SansSerif-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_SansSerif-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-sans-B; src: local('MathJax_SansSerif Bold'), local('MathJax_SansSerif-Bold')}
@font-face {font-family: MJXc-TeX-sans-Bx; src: local('MathJax_SansSerif'); font-weight: bold}
@font-face {font-family: MJXc-TeX-sans-Bw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_SansSerif-Bold.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_SansSerif-Bold.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_SansSerif-Bold.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-sans-I; src: local('MathJax_SansSerif Italic'), local('MathJax_SansSerif-Italic')}
@font-face {font-family: MJXc-TeX-sans-Ix; src: local('MathJax_SansSerif'); font-style: italic}
@font-face {font-family: MJXc-TeX-sans-Iw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_SansSerif-Italic.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_SansSerif-Italic.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_SansSerif-Italic.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-script-R; src: local('MathJax_Script'), local('MathJax_Script-Regular')}
@font-face {font-family: MJXc-TeX-script-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Script-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Script-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Script-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-type-R; src: local('MathJax_Typewriter'), local('MathJax_Typewriter-Regular')}
@font-face {font-family: MJXc-TeX-type-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Typewriter-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Typewriter-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Typewriter-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-cal-R; src: local('MathJax_Caligraphic'), local('MathJax_Caligraphic-Regular')}
@font-face {font-family: MJXc-TeX-cal-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Caligraphic-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-main-B; src: local('MathJax_Main Bold'), local('MathJax_Main-Bold')}
@font-face {font-family: MJXc-TeX-main-Bx; src: local('MathJax_Main'); font-weight: bold}
@font-face {font-family: MJXc-TeX-main-Bw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Main-Bold.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-main-I; src: local('MathJax_Main Italic'), local('MathJax_Main-Italic')}
@font-face {font-family: MJXc-TeX-main-Ix; src: local('MathJax_Main'); font-style: italic}
@font-face {font-family: MJXc-TeX-main-Iw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Main-Italic.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-main-R; src: local('MathJax_Main'), local('MathJax_Main-Regular')}
@font-face {font-family: MJXc-TeX-main-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Main-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-math-I; src: local('MathJax_Math Italic'), local('MathJax_Math-Italic')}
@font-face {font-family: MJXc-TeX-math-Ix; src: local('MathJax_Math'); font-style: italic}
@font-face {font-family: MJXc-TeX-math-Iw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Math-Italic.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-size1-R; src: local('MathJax_Size1'), local('MathJax_Size1-Regular')}
@font-face {font-family: MJXc-TeX-size1-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Size1-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-size2-R; src: local('MathJax_Size2'), local('MathJax_Size2-Regular')}
@font-face {font-family: MJXc-TeX-size2-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Size2-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-size3-R; src: local('MathJax_Size3'), local('MathJax_Size3-Regular')}
@font-face {font-family: MJXc-TeX-size3-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Size3-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-size4-R; src: local('MathJax_Size4'), local('MathJax_Size4-Regular')}
@font-face {font-family: MJXc-TeX-size4-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Size4-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-vec-R; src: local('MathJax_Vector'), local('MathJax_Vector-Regular')}
@font-face {font-family: MJXc-TeX-vec-Rw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Vector-Regular.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Vector-Regular.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Vector-Regular.otf') format('opentype')}
@font-face {font-family: MJXc-TeX-vec-B; src: local('MathJax_Vector Bold'), local('MathJax_Vector-Bold')}
@font-face {font-family: MJXc-TeX-vec-Bx; src: local('MathJax_Vector'); font-weight: bold}
@font-face {font-family: MJXc-TeX-vec-Bw; src /*1*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/eot/MathJax_Vector-Bold.eot'); src /*2*/: url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/woff/MathJax_Vector-Bold.woff') format('woff'), url('https://plato.stanford.edu/archives/win2019/MathJax/fonts/HTML-CSS/TeX/otf/MathJax_Vector-Bold.otf') format('opentype')}
</style></head>

<!-- NOTE: The nojs class is removed from the page if javascript is enabled. Otherwise, it drives the display when there is no javascript. -->
<body class="archive article" id="pagetopright"><div id="MathJax_Message" style="display: none;"></div>
<div id="container">
<div id="header-wrapper">
  <div id="header">
    <div id="branding">
      <div id="site-logo"><a href="../../index.html"><img src="../../symbols/sep-man-red.png" alt="SEP logo" /></a></div>
      <div id="site-title"><a href="../../index.html">Stanford Encyclopedia of Philosophy Archive<div id="site-subtitle">Winter 2019 Edition</div></a></div>
    </div>
    <div id="navigation">
      <div class="navbar">
        <div class="navbar-inner">
          <div class="container">
            <button class="btn btn-navbar collapsed" data-target=".collapse-main-menu" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Menu </button>
            <div class="nav-collapse collapse-main-menu collapse">
              <ul class="nav">
                <li class="dropdown"><a id="drop1" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-book"></i> Browse</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop1">
                    <li><a href="../../contents.html">Table of Contents</a></li>
                    <li><a href="../../new.html">New in this Archive</a></li>
                    
                    <li><a href="../../published.html">Chronological</a></li>
                    <li><a href="../../../../archives/">Archives <i class="icon-external-link"></i></a></li>
                  </ul>
                </li>
                <li class="dropdown"><a id="drop2" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-info-sign"></i> About</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop2">
                    <li><a href="../../info.html">Editorial Information</a></li>
                    <li><a href="../../about.html">About the SEP</a></li>
                    <li><a href="../../board.html">Editorial Board</a></li>
                    <li><a href="../../cite.html">How to Cite the SEP</a></li>
                    <li><a href="../../special-characters.html">Special Characters</a></li>
                    
                    <li><a href="../../../../contact.html">Contact <i class="icon-external-link"></i></a></li>
                  </ul>
                </li>
                <li class="dropdown"><a id="drop3" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-leaf"></i> Support SEP</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop3">
                    <li><a href="../../../../support/">Support the SEP</a></li>
                    <li><a href="../../../../support/friends.html">PDFs for SEP Friends</a></li>
                    <li><a href="../../../../support/donate.html">Make a Donation</a></li>
                    <li><a href="../../../../support/sepia.html">SEPIA for Libraries</a></li>
                  </ul>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
    <!-- End navigation -->
    
    <div id="search">
      <form id="search-form" method="get" action="../../../../search/searcher.py">
        <input type="search" name="query" placeholder="Search this archive" />
<input type="hidden" name="archive" value="win2019" />

        <div class="search-btn-wrapper"><button class="btn search-btn" type="submit"><i class="icon-search"></i></button></div>
      </form>
    </div>
    <!-- End search --> 
    
  </div>
  <!-- End header --> 
</div>
<!-- End header wrapper -->

<div id="content">

<!-- Begin article sidebar -->
<div id="article-sidebar" class="sticky" style="z-index: 999;">
  <div class="navbar">
    <div class="navbar-inner">
      <div class="container">
        <button class="btn btn-navbar collapsed" data-target=".collapse-sidebar" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Entry Navigation </button>
        <div id="article-nav" class="nav-collapse collapse-sidebar collapse">
          <ul class="nav">
            <li><a href="#toc">Entry Contents</a></li>
            <li><a href="#Bib">Bibliography</a></li>
            <li><a href="#Aca">Academic Tools</a></li>
            <li><a href="https://leibniz.stanford.edu/friends/preview/replication/">Friends PDF Preview <i class="icon-external-link"></i></a></li>
            <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/archinfo.cgi?entry=replication&amp;archive=win2019">Author and Citation Info <i class="icon-external-link"></i></a> </li>
            <li><a href="#pagetopright" class="back-to-top">Back to Top <i class="icon-angle-up icon2x"></i></a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div><div></div>
<!-- End article sidebar --> 

<!-- NOTE: Article content must have two wrapper divs: id="article" and id="article-content" -->
<div id="article">
<div id="article-content">

<!-- BEGIN ARTICLE HTML -->


<div id="aueditable"><!--DO NOT MODIFY THIS LINE AND ABOVE-->

<h1>Replication and Reproduction</h1><div id="pubinfo"><em>First published Wed Dec 5, 2001; substantive revision Tue Sep 25, 2018</em></div>

<div id="preamble">

<p>
The problem of replication and reproduction arises out of the history
of genetics [see the entry
 <a href="../gene/">gene</a>
 for a historical review]. It is tied to the concept of the gene and
its generalization in an evolutionary context [see the entry
 <a href="../evolution/">evolution</a>].
 Richard Dawkins introduced the notion of replicators—things
that self-replicate—as a universalization of evolutionary
understandings of genes. Dawkins argued that replicators are the
<em>sine qua non</em> of evolution by natural selection [see the entry
 <a href="../natural-selection/">natural selection</a>],
 while other accounts only require <em>reproduction</em> as one of its
defining features. What exactly is a replicator? How are replicators
different from genes? Can evolution by natural selection occur without
the existence of replicators? Besides the biological domain, are there
any other domains in which replicators have been postulated? To answer
these questions, we will first provide some background for
Dawkins’ notion of replicator and its ties with the concepts of
the gene and information. We will then introduce the distinction
between <em>Replicators</em> and <em>Vehicles</em> in the context of
biological evolution and followed by the extension of this to other
domains. Finally, we will discuss some of the challenges to the idea
that replicators are necessary conditions for evolution by natural
selection.</p>
</div>

<div id="toc">
<!--Entry Contents-->
<ul>
<li><a href="#Back">1. Background</a></li>
<li><a href="#GeneInfo">2. Genes and Information</a></li>
<li><a href="#DawkView">3. Dawkins’ View</a>
   <ul>
   <li><a href="#GeneRepl">3.1 Genes as Replicators</a></li>
   <li><a href="#HullInte">3.2 Hull’s Interactors</a></li>
   </ul></li>
<li><a href="#OtheExamRepl">4. Other Examples of Replicators</a>
   <ul>
   <li><a href="#ImmuSyst">4.1 The Immune System</a></li>
   <li><a href="#SociEvol">4.2 Sociocultural Evolution</a></li>
   <li><a href="#ExteRepl">4.3 The Extended Replicator</a></li>
   </ul></li>
<li><a href="#ChalRepl">5. Challenges to the Replicator</a>
   <ul>
   <li><a href="#DeveSystTheo">5.1 Developmental Systems Theory</a></li>
   <li><a href="#EvolNatuSeleWithRepl">5.2 Evolution by Natural Selection without Replication</a></li>
   <li><a href="#OrigRepl">5.3 Origins of Replicators</a></li>
   <li><a href="#Repr">5.4 Reproducers</a></li>
   </ul></li>
<li><a href="#Conc">6. Conclusion</a></li>
<li><a href="#Bib">Bibliography</a></li>
<li><a href="#Aca">Academic Tools</a></li>
<li><a href="#Oth">Other Internet Resources</a></li>
<li><a href="#Rel">Related Entries</a></li>
</ul>
<!--Entry Contents-->

<hr />
</div>

<div id="main-text">

<h2 id="Back">1. Background</h2>

<p>
The notion of replication arose as a concern in the period immediately
after the second world war (see
 <a href="#fig1">Figure 1</a>),
 with the rise of information theory, communication theory,
cybernetics, and Erwin Schrödinger’s 1944 book <em>What is
Life?</em>. Biologists immediately began to consider the informational
aspect of life, in each of these theoretical frameworks, in areas
ranging from neurology to molecular biology, but especially the latter
(Brookes 1956; Eigen &amp; de Maeyer 1966; Gatlin 1972; Johnson 1970;
Linschitz 1953; Margalef 1957 [1958]; Quastler 1953; Rashevsky 1955).
It was only after the development of communications theory by Shannon
(1948) and cybernetics by Wiener (1948) after it, that inheritance in
biology took on a specifically informational tint (Griffiths &amp;
Stotz 2007). This is the case in particular of the work of Watson and
Crick on the double helix structure of DNA (Watson &amp; Crick 1953),
as well as Crick’s Central Dogma, which states that information
cannot flow from protein to DNA. The Central Dogma (Crick 1958) was
initially cast in terms of structural information (Šustar
2007),
 <a href="#fig2">Figure 2</a>,
 and later recast by Crick as a more cybernetic or computational kind
of information (Crick 1970). [See Eugene V. Koonin 2012 &amp; 2015 for
discussion of the nature of the Central Dogma in current biology.]</p>

<div class="figure avoid-break" id="fig1">
<img src="Ngram_replication.png" alt="a graph with a y axis ranging from 0% to 0.000160% and an x axis from 1800 to 2000; line stays below 0.000020% until 1960 at which point it rapidly rises to hit 0.000130% at 2000" />

<p>
<span class="figlabel">Figure 1.</span> Yearly frequency of count for
the word ‘replication’ in Google text’s English
corpus from 1800–2000 with smoothing = 3. Data obtained from
Google Ngram Viewer (see
 <a href="#Oth">Other Internet Resources</a>).</p>
 </div>

<p>
Structural information is effectively based on the transfer of
molecular primary sequences (linear sequence) in the synthesis of
polypeptides; that is, it is the literal sequence of basic groups
(amino acids), before the protein folds into secondary and higher
conformations. This is a physical and causal account of information:
the new molecule has its sequence because it was templated in some way
by the “information”-imparting molecule (in this case, DNA
to proteins). Griffiths and Stotz (see below) call this, accordingly,
“Crick information”.</p>

<p>
However, in the late 1950s, after the structure of DNA had become more
widely understood, chemist Homer Jacobson proposed a
<em>cybernetic</em> model of reproduction (Jacobson 1958), going so
far as to make a literal model out of train sets and electronic
circuits. Later, von Neumann proposed a theory of self-reproducing
automata (Neumann 1966) which has become a parallel but influential
research program to the issue in biology (Mitchell 2009). Jacob (1973)
cemented the logical description of heredity, but the ultimate
classical source of the notion that genes are information comes from
Williams’ classic 1966 book <em>Adaptation and Natural
Selection,</em> in which he referred to genes as “cybernetic
abstractions” (1966: 33).</p>

<p>
The cybernetic notion of reproduction (and later, replication) rested
on the notion that genes were <em>controllers</em> of phenotypes;
later, this came to be identified (e.g., by Gatlin) as synonymous with
the “communication” conception of information that Shannon
proposed. In biology, genes came to be seen as “programs”
for phenotypes [‘senders’] that passed themselves on to
progeny [‘receivers’], around 1961 (Peluffo 2015).</p>

<p>
Williams’ book was one inspiration for Dawkins’
<em>Selfish Gene</em> (1976), and much of what Dawkins proposed about
replicators is an extension of Williams’ views, in conjunction
with those of William D. Hamilton (1964) and various ethological
ideas. This view is now known as the gene’s eyes view (for a
review see Okasha 2006, Chapter 5)</p>

<p>
Williams introduced the notion of a gene as a substrate neutral
entity:</p>

<blockquote>

<p>
I use the term ‘gene’ to mean that which separates and
recombines with appreciable frequency, (1966: 20)</p>
</blockquote>

<p>
and he further defined an <em>evolutionary</em> gene (that is, one
that is selectively important):</p>

<blockquote>

<p>
[A] gene could be defined as any hereditary information for which
there is a favorable or unfavorable selection bias equal to several or
many times its rate of endogenous change. (1966: 25)</p>
</blockquote>

<p>
His answer to the question of what a gene is, is that a gene is
evolutionary just insofar as it is something heritable that is subject
to rates of selection that exceed the mutation rate. This is a
definition, not a discovery. If it’s favorably or unfavorably
selected and heritable, it’s an evolutionary gene. According to
this definition, items that are not made of DNA can be classed as
evolutionary genes so long as they match Williams’ definition
(Lu &amp; Bourrat 2018).</p>

<div class="figure avoid-break" id="fig2">
<img src="centraldogma.png" alt="a rectangle divided into three horizontal regions, the first region has a purple background and is titled 'Replication' and has a diagram with the word 'DNA' and an arrow going from 'DNA' and circling back to it; the second region has a yellow background and is titled 'Transcription' and the diagram continues with an arrow from 'DNA' in the previous region to the word 'RNA' in this region; the third region has a green background and is titled 'Translation' and the diagram continues with an arrow from 'RNA' in the previous region going into this region and ending." />

<p>
<span class="figlabel">Figure 2.</span> The Central Dogma of genetics
as envisaged by Crick around 1965 (Thieffry &amp; Sarkar 1998).</p>
</div>

<h2 id="GeneInfo">2. Genes and Information</h2>

<p>
The language of “codes” and “information”
flows easily enough with respect to replication (see the entry
 <a href="../information-biological/">biological information</a>).
 Transcription, translation, punctuation, redundancy, synonymy,
editing, proofreading, errors, repairing of errors, messages, copies,
and information all sound natural enough. Yet, the literature dealing
with information is both extensive and factious. Several different
formal analyses of information can be found and very little agreement
about which analysis is best for which subjects. On one point these
scholars tend to agree—cybernetic information and
communication-theoretic information will not do for replication in
biological contexts. Some argue for semantic information
(Godfrey-Smith 2000a; Sarkar 2000; Sterelny 2000b). The trouble is
that no widely accepted version of semantic information exists. Winnie
(2000) distinguishes between Classical and Algorithmic Information
Theory and opts for a revised version of the Algorithmic Theory. But
once again, the problem is that no such formal analysis currently
exists.</p>

<p>
In the face of all this disagreement and unfinished business,
biologists such as Maynard Smith (2000) maintain either that informal
analyses of “information” are good enough or that some
future formal version of information theory will justify the sorts of
inferences that they make. The sense of “information” used
in the Central Dogma of molecular biology is more like a fit of
template, or the primary structure of the protein sequence compared to
the sequence of the DNA base pairs. Attempts have been made in what is
now known as bioinformatics to use Classical Information Theory
(Shannon’s theory of communication) to extract functional and
phylogenetic information (Brooks &amp; Wiley 1988; Gatlin 1972;
Maclaurin 1998; Rodrick Wallace &amp; Wallace 1998), but they appear
to have been unsuccessful in the main.</p>

<p>
While the most likely conclusion is that no version of information
theory as currently formulated can handle “information” as
it functions in biology (Griffiths 2001), attempts have continued to
be made to formulate just such a version (Bergstrom &amp; Rosvall
2011; Sternberg 2008). However, this undercuts the motivation for
appealing to information theory to elucidate genes in the first
instance. Griffiths &amp; Stotz (2013: 153) have argued for
“Crick” information, which they define as “the
specification of the order of amino acids in a polypeptide
chain”. This specificity definition is more chemical than
abstract (see the section below on 
<a href="#DeveSystTheo">Developmental Systems Theory</a>).</p>

<p>
The “informational gene” conception, as James R. Griesemer
(2005) terms it, is a highly abstract concept of a process and the
entities that comprise it. This is intentional, as it permits
Darwinian dynamics to be applied to a range of phenomena, such as
sociocultural, cybernetic, and medical processes. However, this raises
a metaphysical concern, namely that the view that a gene is nothing
but information is a version of Aristotelian hylomorphism (see the
entry
 <a href="../form-matter/">form versus matter</a>).
 That is, information is a type of form, which can be used to explain
physical processes. Classical hylomorphism was roundly demolished as a
<em>scientific</em> hypothesis when Daltonian elements were named and
investigated in the nineteenth century. By 1900, terms like
“substance” (for matter that is propertyless apart from
mass and extension in space) and “form” had taken on a
largely philosophical sense that differed extensively from
Aristotle’s own physical views. Griesemer notes:</p>

<blockquote>

<p>
We know that the informational genes are tied to matter and structure,
but if evolutionary theory – to be general enough to cover
cultural and conceptual change – must be devoid of all reference
to concrete mechanism, it cannot follow from the theory, for example,
that genes are inside organisms or are even parts of organisms, as
Dawkins’s language suggests. Strictly, only the correlations
between replicator and vehicle due to causal connections of a
completely unspecified sort can be implied by such a theory. Striving
to get matter and specific structure out of the theory in order to
make it apply to immaterial realms may thus leave it bankrupt as an
account of causal connection for the material, biological cases.
(Griesemer 2005: 79)</p>
</blockquote>

<p>
And yet, this kind of hylomorphism has remained, even in science.
Biologists have argued that form determined many properties of
organisms in ways that could not be reduced to the actions of their
parts, and this kind of thinking remained and was co-opted by the
molecular biologists and geneticists. Thus, we get the Central Dogma
of molecular biology:</p>

<blockquote>

<p>
The Central Dogma
<br />
This states that once ‘information’ has passed into
protein <em>it cannot get out again</em>. In more detail, the transfer
of information from nucleic acid to nucleic acid, or from nucleic acid
to protein may be possible, but transfer from protein to protein, or
from protein to nucleic acid is impossible. Information means here the
<em>precise</em> determination of sequence, either of bases in the
nucleic acid or of amino acid residues in the protein. (Crick
1958)</p>
</blockquote>

<p>
Read physically, this means only that the structure of the DNA
molecule is not reproduced from the structure of proteins, a perfectly
reasonable account of the molecular processes. But because Crick used
the word “information”, some scientists, including
Dawkins, took this to mean genes are informational entities, that
“code” for organismic traits from the molecular level up
to the entire organism and even beyond. This is striking in a purple
passage of <em>The Blind Watchmaker</em>, where Dawkins describe genes
the following way</p>

<blockquote>

<p>
It is raining DNA outside. [downy seeds from willow trees] …
The cotton wool is mostly made of cellulose, and it dwarfs the tiny
capsule that contains the DNA, the genetic information. The DNA
content must be a small proportion of the total, so why did I say that
it was raining DNA rather than cellulose? The answer is that it is the
DNA that matters… whose coded characters spell out specific
instructions for building willow trees… It is raining
instructions out there, it’s raining programs; it’s
raining tree-growing, fluff spreading, algorithms. That is not a
metaphor, it is the plain truth. It couldn’t be any plainer if
it were raining floppy disks. (1986: 111)</p>
</blockquote>

<p>
So we now turn to Dawkins’ view of genes as replicators, and the
way in which he takes information to be cybernetic and physical at the
same time.</p>

<h2 id="DawkView">3. Dawkins’ View</h2>

<p>
Richard Dawkins introduced the distinction between replicators and
vehicles in his <em>The Selfish Gene</em> (1976). For his purposes
Dawkins found the contrast between genes and organisms too restrictive
and specific. Everyone agrees that genes are replicators, but genes
may not be the only replicators. Dawkins also argued that perhaps more
inclusive entities than single genes might also function as
replicators. At the very least, the possibility should not be ignored.
So, Dawkins adopted “replicator” as a more inclusive and
general term than “gene”. In <em>The Extended
Phenotype</em> he defined a replicator as “anything in the
universe of which copies are made” (Dawkins 1982b: 83). He also
introduced the term “vehicle” for those entities produced
by replicators that help these replicators increase in numbers by
interacting effectively with their environments. This distinction can
be expressed in terms of either entities or processes. According to
Dawkins, replicators function in replication, while vehicles function
in environmental interaction.</p>

<h3 id="GeneRepl">3.1 Genes as Replicators</h3>

<p>
A long-standing dispute in evolutionary biology concerns the levels at
which selection can occur (Bourrat 2015c, 2015b, 2016; Brandon 1996;
Kerr &amp; Godfrey-Smith 2002; Keller 1999; Lloyd 1988; Okasha 2006,
2016; Sober &amp; Wilson 1998; Williams 1966) [see the entry on
 <a href="../selection-units/">units and levels of selection</a>].
 Some authors see this dispute as concerning the levels at which
replication can take place. Other authors take the levels of selection
dispute to concern environmental interaction and insist that
environmental interaction can take place at a variety of levels, from
single genes, cells and organisms to colonies, demes and possibly
entire species. Organisms certainly interact with their environments
in ways that bias the transmission of their genes, but then so do
entities that are both less inclusive than entire organisms (e.g.,
sperm cells) and more inclusive (e.g., beehives).</p>

<p>
Dawkins argued that replication in <em>biological</em> evolution
occurs exclusively at the level of the genetic material. The term
“replication” refers first and foremost to copying, and
genes are the self-replicating molecules of biology. Some critics
(e.g., Lewontin 1991: 48) interpreted this to mean that a strand of
DNA placed on a glass slide might start replicating all on its own. Of
course, no biologist has ever held such view. Genes do replicate
themselves, but only with the aid of highly complicated molecular
machinery. Too often, however, the importance of this machinery has
gone unnoticed. To be sure, when we make copies on a photocopier, we
are interested in the texts, figures or just scrawls that appear on
these sheets of paper. We are not interested in how the photocopier
works, even if it does <em>all</em> the work.</p>

<p>
In Dawkins’ early writings, replicators and vehicles played
different but complementary and equally important roles in selection,
but as he honed his view of the evolutionary process, vehicles became
less and less fundamental. Initially, Dawkins was content to dethrone
the organism from its pride of place in biology. It is an important
focus of environmental interaction, but other entities, both below and
above the organismic level, can also function as vehicles. In later
writings, however, Dawkins went even further to argue that phenotypic
traits are what really matter for evolution by natural selection to
occur, and that they can be treated independently of their being
organized into vehicles. More than that, features such as spider webs
should be viewed as part of a spider’s phenotype. Hence, Dawkins
titled his second book <em>The Extended Phenotype</em> (Dawkins
1982b).</p>

<p>
Dawkins never lost his fascination with vehicular adaptations, a
fascination that his critics denigrate as Panglossian adaptationism.
He filled his books with adaptationist scenarios, some more firmly
supported by data than others, but from the perspective of the
structure of evolutionary theory, he held that replicators are much
more important than vehicles. For example, Dawkins argued at some
length that adaptations are always for the good of replicators, not
vehicles (Lloyd 1992), see entry on
 <a href="../selection-units/">units and levels of selection</a>.
 Vehicles <em>exhibit</em> these adaptations, but ultimately all
adaptations must be <em>explicable</em> in terms of changes in gene
frequencies. Thus, it comes as no surprise when Dawkins (1994: 617)
proclaims that he “coined the term ‘vehicle’ not to
praise it but to bury it”. As prevalent as organisms might be,
as definite as the causal roles that they play in selection are,
reference to them can and must be omitted from any perspicuous
characterization of selection in the evolutionary process. Although
Dawkins is far from a genetic <em>determinist</em>, he is certainly a
genetic <em>reductionist</em>. Whether reductionism itself is good or
bad is a moot question (Sarkar 1998).</p>

<p>
According to Dawkins, replicators have three basic
properties—<em>longevity</em>, <em>fecundity</em> and
<em>copy-fidelity</em>. Longevity means longevity of the gene
<em>type</em> in the form of copies through descent (Dawkins 1982b:
84; Hull 1980), although the stability of gene <em>tokens</em> is
included in the definition in <em>The Selfish Gene</em> (1976: 18). No
gene as a physical body lasts all that long. In mitosis, a gene loses
half of its substance at each replication. What endures, he says, is
not the entity itself but the information incorporated in its
structure. It is this information that is copied with such high
fidelity. Mutations do occur, but at very low frequencies. Even so, in
some organisms mutation rates must be too high because mechanisms have
evolved that search out and repair such errors. The genotype is
therefore an informational notion; information is equivalent to
Aristotelian form. The type is the <em>form</em> of the tokens, in
Dawkins’ view.</p>

<p>
One source of variation in genes of sexual organisms supplementing
mutation is crossover. Pairs of homologous chromosomes line up next to
each other at meiosis, crossover and recombine. For stretches of DNA
in which different alleles exist, the result can be a change in
information. Quite obviously, the shorter the stretch of DNA involved,
the less likely that crossover will occur and the message changed.
Dawkins appeals to such dismantling of entities to argue against
organisms functioning as replicators. In sexual organisms, organisms
themselves are torn apart and repeatedly assembled each generation
(Caporael 2003). If long stretches of DNA lack the necessary identity
by descent to function as replicators, then sexual organisms certainly
lack it. However, some other explanation has to be provided for
asexual organisms because they pass on their overall structure largely
unchanged from generation to generation. For example, R. A. Fisher, in
his <em>The Genetical Theory of Natural Selection</em> (Fisher 1930)
considered the entire genetic complement of asexual organisms to be a
single gene, and this view has been repeated from time to time since.
According to Dawkins, genes, and only genes, can function as
replicators in biological evolution. How large these genetic units are
depends on such things as the prevalence of sex, the frequency of
crossover or lateral gene transfers, and the intensity of
selection.</p>

<blockquote>

<p>
If there were sex but no crossing-over, each chromosome would be a
replicator, and we should speak of adaptations as being for the good
of the chromosome. If there is no sex we can, by the same token, treat
the entire genome of an asexual organism as a replicator. But the
organism itself is not a replicator. (Dawkins 1982a: 95)</p>
</blockquote>

<p>
Dawkins offered two reasons for organisms not being able to function
as replicators. The first is the one he uses to delineate evolutionary
genes. As is the case for long stretches of DNA, organisms are too
easily and frequently broken up to be considered units of replication.
A second reason is that they cannot pass on changes in their
structure, although some phenotypic change may result in generational
change. In fact, <em>some</em> epigenetic mechanisms have been shown
to be passed on across generations (Jablonka &amp; Raz 2009; see the
entry on
 <a href="../inheritance-systems/">inheritance systems</a>).
 The amount of DNA that counts as a replicator certainly varies, but
according to Dawkins, nothing more inclusive than the genetic material
functions as replicators in biological evolution.</p>

<p>
In <em>The Selfish Gene</em>, although Dawkins wanted his definition
of gene to be close to that of Williams’ (1966)
“evolutionary gene”, it isn’t quite:</p>

<blockquote>

<p>
A gene is defined as any portion of chromosomal material which
potentially lasts for enough generations to serve as a unit of natural
selection. (Dawkins 1976: 30)</p>
</blockquote>

<p>
Where Williams’ definition is substrate neutral, Dawkins’
is explicitly chromosomal and DNA-based. Where William’s
“gene” refers to <em>any</em> entity, which is the reason
why it is an informational notion, Dawkins’ gene, by contrast,
is tied to DNA; a view later heavily criticized by Stent (1977), an
influential molecular biologist at the time.</p>

<p>
On Dawkins’ account, the limits of genes need not be absolutely
sharp. Nor must all genes be of the same length. The greater the
selection pressure, the smaller the gene. At the most fundamental
level, selection takes place between alternative alleles residing at
the same locus.</p>

<blockquote>

<p>
As far as a gene is concerned, its alleles are its deadly rivals, but
other genes are just a part of its environment, comparable to
temperature, food, and predators, or companions. (Dawkins 1976:
40)</p>
</blockquote>

<p>
Alleles cannot cooperate with each other; only compete. That is where
the “selfish” in “selfish gene” comes in.
According to Dawkins (1976: 95), the selfish gene is not just one
physical bit of DNA. It is “all replicas of a particular bit of
DNA, distributed throughout the world” (for a recent defense of
a similar approach, see Haig 2012). Hence, genes do not form classes
of spatiotemporally unrelated individuals but trees. They must be
replicas. But being a replica is not enough. The linear repetition of
the “same gene” in the form of several hundred copies is
quite common. These replicas, however, do not reside at the same
<em>locus</em>. Although identical in structure, these genes do not
compete with each other in the way that alleles at the same locus can.
In the simplest and most basic sense, alleles compete with alternative
alleles at the same locus. Any other sorts of competition and
cooperation are merely extrapolations from this fundamental sense of
allelic competition. Even though genes may cooperate with each other
in very complicated ways in embryological development, in replication
they can be treated as “separate and distinct”. In
development the effects of genes blend. In replication replicators do
not blend.</p>

<h3 id="HullInte">3.2 Hull’s Interactors</h3>

<p>
Dawkins introduced the general notions of replicator and vehicle so
that selection need not be limited exclusively to gene-based
biological evolution. However, as the preceding discussion indicates,
his later revisions to his general theoretical outlook were influenced
strongly by the traditional perspective of genes and organisms. Genes
contain the information necessary to produce organisms and their
adaptations. Genes “ride around” in and
“guide” organisms. As Dawkins describes them, vehicles are
relatively discrete entities that “house” replicators and
which can be regarded as machines programmed to preserve and propagate
the replicators that ride inside them. Although these terms may be
appropriate for the relations between genes and organisms, they
interfere with a more general analysis of replication and selection.
What really matters in selection is that entities at various levels of
organization interact with their environments in such a way that the
relevant replicators increase in relative frequency. The actual causal
chain that connects replicators and vehicles need not be limited to
development.</p>

<p>
For example, Dawkins argues at some length that genes and only genes
can function as replicators in biological evolution. He adds that
“all adaptations are for the preservation of DNA; DNA itself
just is” (Dawkins 1982a: 45). But DNA itself exhibits
adaptations. Anyone who has spent much time examining the molecular
structure of DNA soon realizes that it is adapted to replicate. In
addition, the proliferation of junk DNA, transposons, and meiotic
drive are three examples in which the only phenotypes that matter are
phenotypic characteristics of genes (Brandon 1996; Sterelny 1996:
388). Dawkins’ characterizations of replicators, vehicles and
the relations between the two are too closely tied to genes, organisms
and development. DNA can certainly replicate itself, but it can also
function as a “vehicle” even though it cannot code for,
ride around in or direct itself. In sum, a more general
characterization of selection is needed, a characterization that does
not assume that the only causal connection between replicators and
vehicles is development from embryo to maturity. Such a generalization
was offered by Hull (1980, 1981, 1988) who proposed that the relevant
notion is “interactor” rather than “vehicle”,
as interactors are causal and active while vehicles are passive
entities.</p>

<h2 id="OtheExamRepl">4. Other Examples of Replicators</h2>

<p>
Although Dawkins finds it desirable to replace genes with replicators
in his general characterization of the evolutionary process, he says
very little about this more general notion in his <em>The Selfish
Gene</em>. Only one chapter (11) is dedicated to “memetic”
evolution. Instead, as we saw, he discusses the special case of genes
as replicators. The primacy of the genetic perspective in the
characterization of replicators is claimed by some (Griesemer 2000b)
to be one of the chief weaknesses not only in Dawkins’
discussions but also in the work of his successors [see the
section below on
 <a href="#Repr">Reproducers</a>]. Hull
and his colleagues (Hull, Langman, &amp; Glenn 2001) claim to be
providing a general notion of replication, adequate to handle all
different sorts of replication, when too often we are simply reading
peculiarities of genetic replication into our general analysis of
replication.</p>

<h3 id="ImmuSyst">4.1 The Immune System</h3>

<p>
Does replication play the same role in other sorts of selection that
it plays in gene-based biological evolution? The reaction of the
immune system to antigens is closest to the standard biological
example of replication and selection (Hull et al. 2001). Certainly immune systems arose through the same process as other
functional systems. The basic structure of the human immune system is
“built into our genes”, and the only genes that count in
evolution are the ones that we received from our ancestors and can
subsequently pass on to our progeny. In Dawkins’ (1982b: 83)
terminology, the genes that code for our immune system are active,
germ-line replicators. They are active because they influence their
probability to be copied; they are germ-line, because they can
potentially have an infinity of descendants.</p>

<p>
However, the genes that function in the reaction of the immune system
to antigens have two peculiarities. First, they incorporate mechanisms
designed to produce very high frequencies of mutation, and second,
none of the genes involved in the functioning of the immune system are
germ-line. The genes that give rise to B-cells for instance are
designed to mutate extensively until one of these cells identifies an
invader as foreign. It then proliferates extensively as it attacks the
invader. As an organism matures, it accumulates more and more of the
B-cells that have been successful in its past. More than this, as the
process of proliferation continues, the strength of the affinities to
binding sites increases. Initially the primary antibodies almost
always exhibit a weak affinity for their targets, but as the reaction
to the antigen continues, these affinities become stronger.</p>

<p>
Within the confines of a single organism, the reaction of the immune
system to antigens has all the characteristics of selection processes,
but when the organism dies all of these adaptations are lost. In some
species, females pass on not only the genes for the basic structure of
their immune systems but also some of the machinery that past
invasions of antigens have produced in her. However, these cells are
rapidly removed from the offspring as it develops its own immune
system. The reaction of the immune system to antigens departs from
gene-based selection in biological evolution in two ways. First, from
the organismal perspective, the genes that function in protecting an
organism from invaders are not germ-line. They are somatic. Second,
the relevant mutation rates are much, much higher in the immune system
than in ordinary gene-based selection. Instead of mechanisms existing
to discover mutations and repair them, mechanisms exist that encourage
mutations—massively so. If the functioning of the immune system
is to count as an adaptive process, then some changes must be made.
One possibility is to clarify what counts as a “germ-line”
replicator and a rejection of the notion that extremely low mutation
rates are inherent in all selection processes [see
the section below on
 <a href="#ChalRepl">Challenges to the Replicator</a>].</p>

<h3 id="SociEvol">4.2 Sociocultural Evolution</h3>

<p>
The temptation to regard cultures and societies as evolving entities
in a way analogous to organism has a long tradition. In <em>The
Selfish Gene</em> (Chapter 11), Dawkins proposes a version of it that
makes use of the concept of the replicator. He chose the word
“meme” for a sociocultural replicator because of its
similarities with the word gene:</p>

<blockquote>

<p>
We need a name for the new replicator, a noun that conveys the idea of
a unit of cultural transmission, or a unit of imitation.
‘Mimeme’ comes from a suitable Greek root, but I want a
monosyllable that sounds a bit like ‘gene’ (Dawkins 1976:
192).</p>
</blockquote>

<p>
The basic idea, underlying Dawkins’ proposal is that since
evolution by natural selection in the biological domain requires
replication, it must also be the case in the sociocultural domain, and
that imitation, at least in many cases, is the way through which
sociocultural entities, the memes, replicate.</p>

<p>
Important differences exist between sociocultural evolution and
biological evolution (Claidière &amp; André 2012) [see
the entry on
 <a href="../evolution-cultural/">cultural evolution</a>].
 For example, Dawkins (1976) admits that he is on shaky ground when it
comes to the high-fidelity copying required of replicators. Memes seem
to get changed much more frequently than genes (see Sterelny (2006)
for a review). In general, however, it is fair to say that the
standards used to evaluate sociocultural transmission are not on par
with those used to evaluate biological transmission. An overly
idealized view of Mendelian genetics is contrasted to a much more
realistic view of cultural change. So the criticism goes, one problem
with memes is that they do not have discrete boundaries, do not all
come in the same size, and in their functioning are strongly
influenced by their environments. Genes, so the critics claim, have
sharp boundaries, are all of the same size, and are immune to
environmental influences. If sociocultural evolution is to be
evaluated fairly, the same level of criticism must be applied to all
putative examples of selection from gene-based selection in biological
evolution and the reaction of the immune system to antigens to the
development of the central nervous system and social learning (Hull et
al. 2001). The collection of articles in Aunger (2000) present the
arguments pro- and against a memetic approach to culture and a more
recent discussion is available in Lewens (2015).</p>

<p>
Dawkins (1976) also places considerable emphasis on human brains as
the “vehicles” for memetic evolution. He defines
“meme” as an entity capable of being transmitted from one
brain to another. Computers are also plausible vehicles for memes, as
well as artifacts more generally which can ‘store’ and
then release information when a copy of the artifact is made by
emulation (<em>mimesis</em>), that is without the presence of a
demonstration. Dawkins’ discussion of memes is, however, marred
by the pervasiveness of the gene-organism perspective. For example, he
defines “replicator” in terms of transmission of
information—memes leaping from brain to brain or from brains to
computers and back again. But memes do not leap from brain to brain or
from computer to computer. Their content is transmitted in a variety
of ways, including books, audiotapes, conversations and the like. As
much as the physical basis changes, the message remains sufficiently
unchanged. All instances of this message are equally memes, not just
the ones residing in human brains and computers.</p>

<p>
Several accounts of memetic replication have been proposed in the
literature. Some consider that there is no replication in cultural
evolution, but that memes are “attractor points” in
culture (Atran 1998). Others consider that the interactor is the meme
itself (Blackmore 1999) or that the meme is the selected cultural
heritable information just as Williams’ “evolutionary
gene” is the selected genetic heritable information, and the
memetic interactor is the repertoire of behaviors it elicits (Wilkins
1998). One view that has been offered several times is that memes are
active neural structures (Aunger 1998, 2002; Delius 1991).</p>

<p>
All the objections to the gene-meme analogy to one side, Dawkins
(1976: 211) finds the chief difference between genetic and memetic
change is that biological evolution is at bottom a war between alleles
residing at the same locus. “Memes seem to have nothing
equivalent to chromosomes and nothing equivalent to alleles”.
First, the usual depiction of alleles residing at the same locus on
homologous chromosomes so central to Mendelian genetics is an
oversimplification, but more importantly, for at least half of the
history of life on earth, replication and selection took place in the
absence of chromosomes, meiosis and the like. If gene-based biological
evolution took place for so long in the absence of the Mendelian
apparatus and still does so in many extant organisms, then demanding
that memetic evolution proceed by this very special and possibly
aberrant sort of inheritance seems too strong. The cost of meiosis
remains a serious problem in ordinary biological evolution. Demanding
that cultural evolution incorporates this same highly problematic
element in its own makeup seems strange in the extreme. If we are to
develop a general analysis of selection, then we must distinguish
between essential and contingent features of this process..</p>

<p>
Numerous evolutionary biologists question how fundamental to selection
the perspective of alleles at a locus actually is. Almost everyone
agrees that evolution involves changes in gene frequencies. However,
few go on to add that evolution is <em>nothing but</em> changes in
gene frequencies. When one looks at evolutionary biology, one
discovers that it involves much more than changes in gene frequencies.
To be successful, memetic evolution must be fleshed out.</p>

<p>
In 2000, Aunger summarized the disputes between proponents and
skeptics of the memetic approach, as revolving around three
levels:</p>

<blockquote>

<ol>

<li>whether culture is properly seen as composed of independently
transmitted information units;</li>

<li>whether these so-called memes have the necessary qualifications to
serve as replicators; and</li>

<li>whether a Darwinian or selectionist approach such as memetics is
the most feasible or desirable form for a science of culture to take.
(Aunger 2000: 11)</li>
</ol>
</blockquote>

<p>
Nearly 20 years after the diagnosis, it seems however that the memetic
approach has, by and large, been abandoned, with some of its insights
incorporated into other evolutionary approaches to culture. Perhaps
the most successful is dual-inheritance theory (Richerson &amp; Boyd
2008), also known as gene-culture coevolution. A very general
characterization of the sociocultural evolution has been attempted by
El Mouden, André, Morin, &amp; Nettle (2014). This approach
which relies on the most abstract formalism used by evolutionary
theorists, namely the Price equation (for an introduction to the Price
equation see Okasha 2006: Chap. 1), highlights some of the differences
between biological and sociocultural evolution, and why a pure memetic
approach seems difficult to implement as a general framework.</p>

<h3 id="ExteRepl">4.3 The Extended Replicator</h3>

<p>
Some of Dawkins’ critics think that genes play too important of
a role in his notion of replication. Replication, so they argue, can
occur at other levels of organization as well. Just as Dawkins
extended the notion of the phenotype, these authors propose to make
the notion of replicators more general as well—to extend the
replicator so to speak. Dawkins introduced his <em>Extended
Phenotype</em> conception for two reasons. First, he wanted to extend
the notion of a “phenotypic trait”. The sort of nest that
a bird builds or the song that it sings can count as phenotypic traits
just as much as the shape of its bill. Second, as mentioned earlier,
Dawkins wanted to break the hold that organisms have over how we
conceptualize the living world. Traits do tend to come bundled into
reasonably discrete entities, but for making inferences about the
evolutionary process, traits can be treated as separate and distinct.
However, as he proceeded, Dawkins eventually decided that extending
the phenotype was not enough. He had to bury it. All sorts of
fascinating mechanisms to one side, what really matters in selection
takes place at the level of replicators.</p>

<p>
Although such critics of Dawkins as Sterelny, Smith, &amp; Dickison
(1996) declined to join with Dawkins in his demotion of environmental
interaction, they did agree with him that replicators are special.
They play a special role in development. However, they do not limit
replicators to genes even in biological evolution. Sterelny et al.
propose to extend the replicator to include nonstandard entities:
“the set of developmental resources that are adapted from the
transmission of similarity across the generations” (Sterelny
2001: 338). For example, the sort of burrow that a particular organism
digs is influenced by its genetic makeup, but if these burrows are
used over and over again, characteristics of these burrows can
themselves be viewed as replicators. The effects of these burrows get
passed on from generation to generation, but not via the DNA.</p>

<p>
As more and more non-DNA replicators, including epigenetic
inheritance, are acknowledged, Dawkins’ view began to grade into
the conception of the Extended Replicator, which Sterelny et al.
(1996) specify as any non-genetic as well as genetic replicators where
‘genetic’ refers to DNA. These two views are so general
that any case that can be described in one can be redescribed in the
other. Differences lie in ease of description. According to Sterelny
et al. (1996), the burrows that some organisms dig can function as
replicators—extended replicators—while Dawkins portrays
them as instance of extended phenotypes. The contrast is between
selfish burrows and selfish genes for burrowing.</p>

<p>
In the version of replication formulated by Sterelny et al. (1996)
both copying and biofunction are crucial. Copying is quite obviously a
causal phenomenon, but not any old causal connection will do.
Similarity of copies is necessary, but not sufficient. These copies
must be copies of copies. One copy must produce another copy and that
copy produce still another copy and so on. For one entity to be a copy
of another, it must be the output of a process whose biofunction is to
conserve function. The function of copying is to produce from one
token another token which is similar in the relevant respects. Genes
fit this definition, but so do lots of examples of non-genetic
transmission; e.g., habitat stability resulting from nest site
imprinting, the song that a bird learns, various microorganism
symbionts, not to mention sociocultural transmission.</p>

<h2 id="ChalRepl">5. Challenges to the Replicator</h2>

<h3 id="DeveSystTheo">5.1 Developmental Systems Theory</h3>

<p>
In the wake of Oyama’s <em>The Ontogeny of Information</em>
(1985, 2000) and other work, notably that of Daniel Lehrman (1953)
and Patrick Bateson (2001), a view of biological evolution has arisen
that emphasizes development (see also Griffiths &amp; Gray 1994a). In
the nineteenth century development was an extremely active research
program. The next great discoveries in biology were going to be in the
area of development. It was not to be. First Mendelian genetics and
then a version of evolutionary theory centered on population genetics
took over biology, and they did so while avoiding development.
Everyone knew that development was central to both evolution and
reproduction, but no one could see how to integrate the masses of
developmental data available into the emerging synthesis of
evolutionary biology and genetics. As a result, development was left
out of the Modern Synthesis (Beurton, Falk, &amp; Rheinberger 2000;
Gilbert 1991). Considering how central development actually is in
metazoan and metaphyte biology, the advances made while ignoring it
are staggering.</p>

<p>
Even so, developmental biology continued on its course until at long
last we seem to understand development well enough to begin
integrating it into the rest of biology. On the most conservative
view, current versions of evolutionary theory can remain largely
unchanged as development is grafted onto them. On a second view, both
perspectives are likely to require some modification to bring off this
integration. Our understanding of development may have to be modified,
but so too for evolutionary theory (Pigliucci &amp; Müller 2010).
Finally, at the other extreme, development will all but replace
evolutionary theory. In their more exuberant moments, some advocates
of Developmental Systems Theory (hereafter DST) seem to be claiming
just that. Just as some molecular biologists think that molecular
biology is rapidly replacing all the rest of biology, some advocates
of DST argue that developmental theories will simply replace current
versions of evolutionary theory.</p>

<p>
On some versions of the DST view, genes have no uniquely privileged
role in repeated cycles of development (Griffiths 2001; Griffiths
&amp; Gray 1994b). In fact, no element of the developmental matrix
plays any privileged causal role—not genes, not organisms, not
the environment, not anything. Everything counts as a resource, albeit
in particular situations certain resources will play more important
roles than other resources. In rejecting any privileged role for
genes, advocates of DST are especially skeptical of one particular
role supposedly played by genes—the transmission of information.
According to some, information is central to developmentalism, but
genes are not the only mechanisms for information transfer. According
to others, information plays no role in the emerging developmentalist
perspective. The developmental system as a whole is the unit of
selection [see the entry on
 <a href="../selection-units/">units and levels of selection</a>].
 If genes have no special role and whole developmental cycles are
replicators, as some developmental- systems theorists argue, the
distinction between replicator and interactor becomes blurred .</p>

<p>
In the continuing debate over developmental versus traditional
theories of evolutionary biology, Sterelny et al. (1996) hold a fairly
conservative position. They agree with the developmentalists that
genes play no privileged role in the development of phenotypes from
genotypes. Genes do play a role in this process, only simply not a
<em>privileged</em> role. Genes can serve as a causal bridge from
phenotype to phenotype, but other entities can do so as well. Genes
are not the only replicators in biological evolution. The repeated
cycles in inheritance include many different sorts of constancies and
repetitions—genes, cellular machinery, phenotypic traits
including behaviors, and social structures. Information remains
central to selection processes, but genes are not the only carriers of
such information. Genes predict phenotypic characters only in the same
sense that environmental factors predict them. Sterelny (2000a) has
argued that developmental considerations require no fundamental
reevaluation of evolutionary conceptions along the lines of DST.</p>

<h3 id="EvolNatuSeleWithRepl">5.2 Evolution by Natural Selection without Replication</h3>

<p>
Godfrey-Smith (2009; see also Godfrey-Smith 2000b; Griesemer 2005)
contrasts the replicator/interactor approach to natural selection with
a more classical approach. This latter approach is usually given as a
“recipe” for evolution by natural selection to use
Godfrey-Smith’s terminology (see also Brandon 1990: 3.1). Ever
since Darwin, a number of authors have proposed that for evolution by
natural selection to occur three conditions or ingredients are
necessary. A population should exhibit (1) variation, that (2) leads
to differences in fitness between the entities forming the population
[see the entry on
 <a href="../fitness/">fitness</a>],
 (3) which are heritable (reviewed in Godfrey-Smith 2007). The most
famous version of these recipes is the one provided by Lewontin
1970.</p>

<p>
There is a difference in emphasis between the two approaches in that
the replicator/interactor approach focuses on the <em>entities</em>
forming the population, while the classical approach provides a
population description of evolutionary change. Another difference
between the two approaches, is that under the classical approach
replication is not required for evolution by natural selection to
occur. In fact, as long as heritability is positive, and the two other
conditions are satisfied, then evolution by natural selection can
occur (but will not necessarily so, cf. Godfrey-Smith 2009: 25). An
abstract definition of heritability (<span class="MathJax_Preview" style="color: inherit; display: none;"></span><span id="MathJax-Element-1-Frame" class="mjx-chtml MathJax_CHTML" tabindex="0" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;msup&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;/math&gt;" role="presentation" style="font-size: 101%; position: relative;"><span id="MJXc-Node-1" class="mjx-math" aria-hidden="true"><span id="MJXc-Node-2" class="mjx-mrow"><span id="MJXc-Node-3" class="mjx-msubsup"><span class="mjx-base"><span id="MJXc-Node-4" class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style="padding-top: 0.483em; padding-bottom: 0.303em;">h</span></span></span><span class="mjx-sup" style="font-size: 70.7%; vertical-align: 0.513em; padding-left: 0px; padding-right: 0.071em;"><span id="MJXc-Node-5" class="mjx-mn" style=""><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.363em; padding-bottom: 0.363em;">2</span></span></span></span></span></span><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><msup><mi>h</mi><mn>2</mn></msup></math></span></span><script type="math/tex" id="MathJax-Element-1">h^2</script>)—although by no
means fully general (see Bourrat 2015a)—is that it represents
the slope of the regression of average offspring character on parent
or mid-parent (in case of sexual species) character (see the entry on
 <a href="../heredity/">Heritability</a>).
 Given a positive heritability does not entail replication of
character between parent and offspring, but is rather a special case,
in which heritability would be exactly 1 in the absence of
environmental variation, the replicator/interactor perspective is
subsumed by the classical approach. Godfrey-Smith concurs:</p>

<blockquote>

<p>
From this point of view, we can see the replicator analysis as picking
out a special case of what is covered (or supposed to be covered) by
the classical view. (2009: 36; see also Godfrey-Smith 2000b)</p>
</blockquote>

<p>
To see why a lack of replication does not prevent evolution by natural
selection from occurring, suppose a population of entities exists with
different sizes, as presented in
 <a href="#fig3a">Figure 3(a)</a>,
 in which there is selection for larger entities. We assume that the
entities reproduce asexually in discrete generations and that there is
no environmental variation. At the parental generation, there are two
types of individuals, with respective sizes of <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span id="MathJax-Element-2-Frame" class="mjx-chtml MathJax_CHTML" tabindex="0" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mfrac&gt;&lt;mn&gt;3&lt;/mn&gt;&lt;mn&gt;4&lt;/mn&gt;&lt;/mfrac&gt;&lt;/math&gt;" role="presentation" style="font-size: 101%; position: relative;"><span id="MJXc-Node-6" class="mjx-math" aria-hidden="true"><span id="MJXc-Node-7" class="mjx-mrow"><span id="MJXc-Node-8" class="mjx-mfrac"><span class="mjx-box MJXc-stacked" style="width: 0.495em; padding: 0px 0.12em;"><span class="mjx-numerator" style="font-size: 70.7%; width: 0.7em; top: -1.345em;"><span id="MJXc-Node-9" class="mjx-mn" style=""><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.363em; padding-bottom: 0.363em;">3</span></span></span><span class="mjx-denominator" style="font-size: 70.7%; width: 0.7em; bottom: -0.628em;"><span id="MJXc-Node-10" class="mjx-mn" style=""><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.363em; padding-bottom: 0.363em;">4</span></span></span><span class="mjx-line" style="border-bottom: 1.3px solid; top: -0.288em; width: 0.495em;"></span></span><span class="mjx-vsize" style="height: 1.395em; vertical-align: -0.444em;"></span></span></span></span><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfrac><mn>3</mn><mn>4</mn></mfrac></math></span></span><script type="math/tex" id="MathJax-Element-2">\frac{3}{4}</script> and 1.
Because of the selection for size, entities of size <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span id="MathJax-Element-3-Frame" class="mjx-chtml MathJax_CHTML" tabindex="0" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mfrac&gt;&lt;mn&gt;3&lt;/mn&gt;&lt;mn&gt;4&lt;/mn&gt;&lt;/mfrac&gt;&lt;/math&gt;" role="presentation" style="font-size: 101%; position: relative;"><span id="MJXc-Node-11" class="mjx-math" aria-hidden="true"><span id="MJXc-Node-12" class="mjx-mrow"><span id="MJXc-Node-13" class="mjx-mfrac"><span class="mjx-box MJXc-stacked" style="width: 0.495em; padding: 0px 0.12em;"><span class="mjx-numerator" style="font-size: 70.7%; width: 0.7em; top: -1.345em;"><span id="MJXc-Node-14" class="mjx-mn" style=""><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.363em; padding-bottom: 0.363em;">3</span></span></span><span class="mjx-denominator" style="font-size: 70.7%; width: 0.7em; bottom: -0.628em;"><span id="MJXc-Node-15" class="mjx-mn" style=""><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.363em; padding-bottom: 0.363em;">4</span></span></span><span class="mjx-line" style="border-bottom: 1.3px solid; top: -0.288em; width: 0.495em;"></span></span><span class="mjx-vsize" style="height: 1.395em; vertical-align: -0.444em;"></span></span></span></span><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfrac><mn>3</mn><mn>4</mn></mfrac></math></span></span><script type="math/tex" id="MathJax-Element-3">\frac{3}{4}</script>
produce 3 offspring while entities of size 1 produce 4 offspring.
Suppose now that there is a positive heritability, as defined above,
so that, on average, offspring resemble more their parent than they
resemble other entities of the parental generation, as is displayed on
 <a href="#fig3a">Figure 3(a)</a>,
 but only slightly so (<span class="MathJax_Preview" style="color: inherit; display: none;"></span><span id="MathJax-Element-4-Frame" class="mjx-chtml MathJax_CHTML" tabindex="0" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;msup&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;0.2&lt;/mn&gt;&lt;/math&gt;" role="presentation" style="font-size: 101%; position: relative;"><span id="MJXc-Node-16" class="mjx-math" aria-hidden="true"><span id="MJXc-Node-17" class="mjx-mrow"><span id="MJXc-Node-18" class="mjx-msubsup"><span class="mjx-base"><span id="MJXc-Node-19" class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style="padding-top: 0.483em; padding-bottom: 0.303em;">h</span></span></span><span class="mjx-sup" style="font-size: 70.7%; vertical-align: 0.513em; padding-left: 0px; padding-right: 0.071em;"><span id="MJXc-Node-20" class="mjx-mn" style=""><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.363em; padding-bottom: 0.363em;">2</span></span></span></span><span id="MJXc-Node-21" class="mjx-mo MJXc-space3"><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.063em; padding-bottom: 0.303em;">=</span></span><span id="MJXc-Node-22" class="mjx-mn MJXc-space3"><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.363em; padding-bottom: 0.363em;">0.2</span></span></span></span><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><msup><mi>h</mi><mn>2</mn></msup><mo>=</mo><mn>0.2</mn></math></span></span><script type="math/tex" id="MathJax-Element-4">h^2 = 0.2</script>). After one generation,
although there is no high-fidelity replication, the average size of
the population has changed, due to the effect of natural
selection. This simple example, demonstrates quite straightforwardly
that replication is not required for evolution by natural selection to
occur.</p>

<p>
We said earlier that replication would imply a heritability of exactly
1 in the absence of environmental variation. Although this is correct,
the reverse is not true. In fact, take again a similar population as
in
 <a href="#fig3a">Figure 3(a)</a>,
 but this time assume a different pattern of inheritance for the two
types of entities as displayed in
 <a href="#fig3b">Figure 3(b)</a>.
 With this pattern of inheritance, although there is no replication,
entities, on average, have the exact same character as their parent.
In the case of the entities of size <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span id="MathJax-Element-5-Frame" class="mjx-chtml MathJax_CHTML" tabindex="0" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mfrac&gt;&lt;mn&gt;3&lt;/mn&gt;&lt;mn&gt;4&lt;/mn&gt;&lt;/mfrac&gt;&lt;/math&gt;" role="presentation" style="font-size: 101%; position: relative;"><span id="MJXc-Node-23" class="mjx-math" aria-hidden="true"><span id="MJXc-Node-24" class="mjx-mrow"><span id="MJXc-Node-25" class="mjx-mfrac"><span class="mjx-box MJXc-stacked" style="width: 0.495em; padding: 0px 0.12em;"><span class="mjx-numerator" style="font-size: 70.7%; width: 0.7em; top: -1.345em;"><span id="MJXc-Node-26" class="mjx-mn" style=""><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.363em; padding-bottom: 0.363em;">3</span></span></span><span class="mjx-denominator" style="font-size: 70.7%; width: 0.7em; bottom: -0.628em;"><span id="MJXc-Node-27" class="mjx-mn" style=""><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.363em; padding-bottom: 0.363em;">4</span></span></span><span class="mjx-line" style="border-bottom: 1.3px solid; top: -0.288em; width: 0.495em;"></span></span><span class="mjx-vsize" style="height: 1.395em; vertical-align: -0.444em;"></span></span></span></span><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfrac><mn>3</mn><mn>4</mn></mfrac></math></span></span><script type="math/tex" id="MathJax-Element-5">\frac{3}{4}</script>, <em>some</em>
offspring have the exact same character as their parent, but that is
not required either, as is the case with the entities of size 1, of
which none of the offspring has the same character as their
parent.</p>

<p>
One possible response against regarding the classical approach to
evolution by natural selection as unconditionally superior to the
replicator/interactor approach, is to argue that although evolution by
natural selection can occur without replication, when this happens,
the evolutionary change resulting is not a <em>pure</em> case of
evolution by natural selection. Rather, it is evolution by natural
selection mixed with some other evolutionary processes. A rationale
for this claim is that the fact offspring do not resemble their parent
introduces some variation in the population of which the origin is not
natural selection. In abstract terms, the introduction of variation
can be regarded as a form mutation.</p>

<p>
Under this interpretation, the classical approach on the one hand, and
the replicator/interactor approach, on the other hand, just emphasize
different aspects of evolutionary explanations. The classical approach
gives us evolutionary explanations <em>in the context of other
evolutionary processes occurring</em>, while the replicator/interactor
approach gives us evolutionary explanations <em>of what the effect of
natural selection would be in the absence of any other evolutionary
process acting in the population</em>, that is, in its pure form.
Understood that way, the two approaches do not necessarily need to be
opposed.</p>

<h3 id="OrigRepl">5.3 Origins of Replicators</h3>

<p>
Another challenge to the replicator/interactor approach to evolution
by natural selection is to see replicators themselves as a
<em>product</em> of natural selection rather than as a necessary
starting point. One premise of this argument is that some form of
natural selection must have occurred on entities that initially were
not replicating, perhaps not even reproducing. This is recognized by
Dawkins, who notes in passing in the second chapter of <em>the Selfish
Gene</em> that “Darwin’s ‘survival of the
fittest’ is really a special case of a more general law of
survival of the stable” (Dawkins 1976: 13) and that life
originated from selection of molecules for stability. The idea that
evolution by natural selection can occur without reproduction has been
argued by Bouchard (2008, 2011; see also Doolittle 2014, Bourrat
2014).</p>

<div class="figure avoid-break" id="fig3">

<div id="fig3a">
<img src="low_herit.svg" width="350" alt="a diagram with two parts, the first consists of a black circle labeled '0.75' with arrows pointing to three black circles below labeled respectively '0.8', '0.8', and '0.95', below this is a legend stating 'Average: 0.85'. The second part consists of a black circle labeled '1' with arrows pointing to four black circles below labeled respectively '0.8', '0.8', '1', and '1', below is a legend stating 'Average: 0.9'. below all is an equation \(h^2 = 0.2\) " />

<p>
(a) Population in which there is no replication and a low
heritability (<span class="MathJax_Preview" style="color: inherit; display: none;"></span><span id="MathJax-Element-6-Frame" class="mjx-chtml MathJax_CHTML" tabindex="0" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;msup&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;/math&gt;" role="presentation" style="font-size: 101%; position: relative;"><span id="MJXc-Node-28" class="mjx-math" aria-hidden="true"><span id="MJXc-Node-29" class="mjx-mrow"><span id="MJXc-Node-30" class="mjx-msubsup"><span class="mjx-base"><span id="MJXc-Node-31" class="mjx-mi"><span class="mjx-char MJXc-TeX-math-I" style="padding-top: 0.498em; padding-bottom: 0.286em;">h</span></span></span><span class="mjx-sup" style="font-size: 70.7%; vertical-align: 0.513em; padding-left: 0px; padding-right: 0.071em;"><span id="MJXc-Node-32" class="mjx-mn" style=""><span class="mjx-char MJXc-TeX-main-R" style="padding-top: 0.357em; padding-bottom: 0.357em;">2</span></span></span></span></span></span><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><msup><mi>h</mi><mn>2</mn></msup></math></span></span><script type="math/tex" id="MathJax-Element-6">h^2</script>), yet, evolution by natural selection occurs.</p>
</div>

<div id="fig3b">
<img src="high_herit.svg" width="350" alt="a diagram with two parts, the first consists of a black circle labeled '0.75' with arrows pointing to three black circles below labeled respectively '0.5', '0.75', and '1', below this is a legend stating 'Average: 0.75'. The second part consists of a black circle labeled '1' with arrows pointing to four black circles below labeled respectively '0.75', '0.75', '1.25', and '1.25', below is a legend stating 'Average: 1'. below all is an equation \(h^2 = 1\) " />

<p>
(b) Population in which there is no replication, yet heritability 
is 1. Evolution by natural selection occurs.</p>
</div>

<p>
<span class="figlabel">Figure 3.</span> Illustration of the idea that
evolution by natural selection can occur without replication.</p>
</div>

<p>
The idea that selection can occur without there being replicators was
first introduced as “chemical selection” in line with the
views of Oparin, who proposed that coascervates (colloidal droplets)
could encapsulate protobiological chemicals and self-replicate without
genes through mechanical fission, a view also independently proposed
by Haldane (Oparin 1936 [1938], Haldane 1929; see Kolb 2016). This was
later extended by the work of Sidney Fox, who showed that proteinoid
spheres could mechanically reproduce stably (Hsu &amp; Fox 1976).</p>

<p>
If modern biological systems evolved from molecules that did not have
the ability to reproduce, let alone replicate, one might ask, what is
the simplest case of evolution by natural selection? It is a case in
which different types of entities have different persistences but do
not produce any offspring. In this model, the types with the property
of persisting longer represent an increasing frequency of the total
population over time. One problem though in a model of persistence
alone, is that each time one entity goes out of existence, it is not
replaced by another entity, so that over time the population size
tends towards zero. Although this does not mean that evolution by
natural selection cannot occur in such populations, this poses
important constraints on the evolutionary dynamics of this population.
Furthermore, it is possible to regard the ability for a population to
maintain its size or increase its size as a primordial form of
adaptation. To see this one can imagine different populations of
entities at different locations, some with the ability to produce some
entities or multiply, others without. It is clear that the populations
with the ability to maintain or increase their size would out-compete
those that are unable to do so. The ability for a population to
sustain its population size might thus be regarded as a primordial
form of adaptation.</p>

<p>
A next ‘stage’ in the evolution from primordial entities
unable to reproduce to modern biological systems might thus have been
for entities to produce new offspring entities (or multiply) without
the ability to pass on their properties reliably. In such a situation
the size of a population could be maintained. The most extreme form of
that would be to imagine a population of two types in which each type
has the same probability to produce an offspring of the other types
than its own type. With entities of the two types having different
persistence, a modest form of evolution by natural selection is still
possible. Versions of this model have been proposed by Wilkins,
Stanyon, &amp; Musgrave (2012), Earnshaw-Whyte (2012) and Bourrat
(2014). Since there is, on average, no more resemblance between an
offspring with its parent than any other entity of the parental
generation, and evolution by natural selection is still observed in
such a scenario, one might think that heritability is, strictly
speaking not a necessary condition for evolution by natural
selection.</p>

<p>
Wilkins et al. (2012) argue that over time in a reactor with limited
incoming chemical species, hypercycles (as defined by Eigen,
Biebricher, Gebinoga, &amp; Gardiner 1991) of reactions will tend to
include reaction types that are more stable and capture more source
molecules and energy than variant reaction types, leading to the
evolution by natural selection of replicator-like molecules as well as
energetically efficient “metabolic” molecules. Recent work
in biochemistry has added strength to that suggestion (Altstein 2015).
Hence, given that no replicator is required for selection (in this
case, chemical), replicator molecules can be seen as the
<em>outcome</em> of selective evolution, not a necessary prior
condition for it. In effect, the evolution of replicators is the first
major transition of evolution, despite the view of Szathmáry
and Maynard Smith that replicators needed to come first (Maynard Smith
&amp; Szathmáry 1995; Szathmáry &amp; Maynard Smith
1997). Bourrat (2014), further developed this idea into a series of
agent-based models. He showed that replication is an attractor in
populations lacking heritability once small random mutations
increasing or decreasing the fidelity of character inheritance is
introduced.</p>

<h3 id="Repr">5.4 Reproducers</h3>

<p>
The works on the origins of replicators ties in with the notion of the
<em>reproducer</em> proposed by Griesemer (2000a,b, 2002, 2005;
Wimsatt &amp; Griesemer 2007), although it was published first, based
on a circulated manuscript by Griesemer, by Szathmáry and
Maynard Smith (1997). A reproducer is an entity that develops and has
a material overlap between the “parent” and
“progeny”. This causal relationship between parents and
progeny is called “progeneration”, the “increase in
numerically distinct objects of a given kind”. Reproduction is a
composite process of development and progeneration, leading to objects
that are themselves capable of development and progeneration. Thus,
the reproducer concept extends the notion of a developmental
system.</p>

<p>
Here inheritance is a <em>system</em> property, not a property of
parts, unless they are also reproducing systems, and is
developmentally acquired. Organisms do not arise already capable of
reproduction in the main, but must undergo orderly transitions before
they may reproduce.</p>

<p>
The relation between reproducers, and replicators and interactors,
varies according to author. Griesemer treats replication as the
terminal form of reproduction, once coding mechanisms have been
evolved, which is itself an evolved form of multiplication (Griesemer
2000a: 76). Unlike Dawkins’ replicator concept, which Griesemer
believes is based on a similarity relationship of
“copying”, a reproducer requires <em>material overlap</em>
of systems. However, replication arises when material overlap evolves
progeneration, and progeneration evolves a codical inheritance system,
as
 <a href="#fig4">Figure 4</a>
 indicates.</p>

<div class="figure avoid-break" id="fig4">
<img src="replication.svg" width="500" alt="title is 'Modes of Multiplication'. This is a tree diagram. At the top is a node labeled 'Multiplication' below and to the left is a node labeled 'Copying' with the line connecting labeled 'nonmaterial overlap (property transmission)'. Below and to the right is 'Progeneration' with the line connecting labeled 'material overlap (parts transfer)'. Below 'Progeneration' is 'Reproduction' with the line connecting labeled 'material overalp of mechanisms of development'. Below 'Reproduction' is 'Inheritance' with the line connecting labeled 'material overlap of evolved mechanisms of development'. Below 'Inheritance' is 'Replication' with the line connected labeled 'material overlap of evolving coding mechanisms of development'. " />

<p>
<span class="figlabel">Figure 4.</span> Wimsatt and Griesemer’s
reproducer concept (Wimsatt &amp; Griesemer 2007, 268).</p>
</div>

<p>
Others, particularly Jablonka and Lamb (2005, 2007), think that
reproducers are distinct from replication, and that some epigenetic
inheritance systems are reproducers without being replicators. For
them, a reproducer is the “target of selection” (see the
entry on
 <a href="../inheritance-systems/">inheritance systems</a>).
</p>

<p>
Godfrey-Smith (2009: 83; see also Okasha 2006: 14) takes issue with
Griesemer’s requirement of “material overlap” as
being too narrow a definiendum of reproducers. He thinks that
reproducers need not contribute actual material to their progeny
(where they template for sequences, as in the case of viruses).
Griesemer believes Godfrey-Smith has over-interpreted the requirement;
he is primarily concerned with the idea that parents <em>cause</em>
progeny (Griesemer 2014). Charbonneau (2014) furthers the move made by
Godfrey-Smith. He argues that the existence of parent-offspring
lineages is unnecessary for evolution by natural selection to occur.</p>

<p>
An aspect of reproducers <em>sensu lato</em> that has fascinated some
is that it does not imply a particular level of reproduction or
progeneration, and indeed Szathmáry and Maynard Smith consider
that the major transitions of evolution each involve an evolved mode
of progeneration that is more inclusive or novel than what it evolved
from (Maynard Smith &amp; Szathmáry 1995; Szathmáry
&amp; Maynard Smith 1997).</p>

<p>
Godfrey-Smith also proposed a taxonomy of reproducers, dividing them
into simple, collective and scaffolded (Godfrey-Smith 2009, 87ff).
Simple reproducers are </p>

<blockquote>

<p>
entities that can reproduce … using their own machinery, in
conjunction with external sources of energy and raw materials. </p>
</blockquote>

<p>
He gives an example of a bacterial cell. Collective reproducers </p>

<blockquote>

<p>
… are reproducing entities with parts that themselves have the
capacity to reproduce, where the parts do so largely through their own
resources rather than the coordinated activity as a whole. </p>
</blockquote>

<p>
And scaffolded reproducers </p>

<blockquote>

<p>
are entities which get reproduced as part of the reproduction of some
larger unit (a simple reproducer), or that are reproduced by some
other entity. </p>
</blockquote>

<p>
Examples here includes viruses and chromosomes. Godfrey-Smith applies
the scaffolded reproducer category to sociocultural evolution.</p>

<p>
The notion of reproducers has also been applied to speciation and
cladogenesis (A. Hamilton &amp; Haber 2006). While it may apply to
populations, given that taxa do not share the kind of causal unity
found in populations and organisms, it is unlikely to apply to species
or supraspecific taxa, and macroevolution in general.</p>

<h2 id="Conc">6. Conclusion</h2>

<p>
The idea that evolution by natural selection requires the existence of
replicators has been a very useful heuristics in evolutionary biology,
but also a way of motivating the application of an evolutionary
thinking beyond the boundaries of evolutionary biology. We started by
showing how the notion of gene can be abstracted away from its
physical basis and characterized in purely informational terms. We
then showed how Dawkins and Hull pursued this abstraction and
attempted to free the gene from its biological substrate. This
motivated responses both from proponents of the replicator concept who
furthered the abstraction but also critics. As successful and
inspiring such an abstraction has been, it should not be reified:
evolution by natural selection and adaptation can occur even in the
absence of replicators. High fidelity certainly facilitates adaptive
evolution, but this might be because replicators are themselves the
product of evolution rather than necessary for it.</p>
</div>

<div id="bibliography">

<h2 id="Bib">Bibliography</h2>

<ul class="hanging">

<li>Altstein, Anatoly D., 2015, “The Progene Hypothesis: The
Nucleoprotein World and How Life Began”, <em>Biology
Direct</em>, 10(1): 67. doi:10.1186/s13062-015-0096-z </li>

<li>Atran, Scott, 1998, “Folk Biology and the Anthropology of
Science: Cognitive Universals and the Cultural Particulars”,
<em>Behavioral and Brain Sciences</em>, 21(4): 547–609.
doi:10.1017/S0140525X98001277</li>

<li>Aunger, Robert, 1998, “The‘Core Meme’
Meme”, <em>Behavioral and Brain Sciences</em>, 21(4):
569–570. doi:10.1017/S0140525X98221273</li>

<li>–––, 2000, <em>Darwinizing Culture: The Status
of Memetics as a Science</em>, Oxford: Oxford University Press.
doi:10.1093/acprof:oso/9780192632449.001.0001</li>

<li>–––, 2002, <em>The Electric Meme: A New Theory
of How We Think</em>, New York: Free Press.</li>

<li>Bateson, Patrick, 2001, “Behavioral Development and
Darwinian Evolution”, in Oyama et al. 2001: 149–166.</li>

<li>Bergstrom, Carl and Martin Rosvall, 2011, “The Transmission
Sense of Information”, <em>Biology and Philosophy</em>, 26(2):
159–176. doi:10.1007/s10539-009-9180-z </li>

<li>Beurton, Peter, Raphael Falk, and Hans-Jörg Rheinberger
(eds), 2000, <em>The Concept of the Gene in Development and Evolution:
Historical and Epistemological Perspectives</em>, Cambridge: Cambridge
University Press. doi:10.1017/CBO9780511527296</li>

<li>Blackmore, Susan J., 1999, <em>The Meme Machine</em>, Oxford; New
York: Oxford University Press.</li>

<li>Bouchard, Frédéric, 2008, “Causal Processes,
Fitness, and the Differential Persistence of Lineages”,
<em>Philosophy of Science</em>, 75(5): 560–570.
doi:10.1086/594507</li>

<li>–––, 2011, “Darwinism Without Populations:
A More Inclusive Understanding of the ‘Survival of the
Fittest’”, <em>Studies in History and Philosophy of
Science Part C: Studies in History and Philosophy of Biological and
Biomedical Sciences</em>, 42(1): 106–114.
doi:10.1016/j.shpsc.2010.11.002</li>

<li>Bourrat, Pierrick, 2014, “From Survivors to Replicators:
Evolution by Natural Selection Revisited”, <em>Biology and
Philosophy</em>, 29(4): 517–538.
doi:10.1007/s10539-013-9383-1</li>

<li>–––, 2015a, “How to Read
‘Heritability’ in the Recipe Approach to Natural
Selection”, <em>The British Journal for the Philosophy of
Science</em>, 66(4): 883–903. doi:10.1093/bjps/axu015 </li>

<li>–––, 2015b, “Levels of Selection Are
Artefacts of Different Fitness Temporal Measures”,
<em>Ratio</em>, 28(1): 40–50. doi:10.1111/rati.12053 </li>

<li>–––, 2015c, “Levels, Time and Fitness in
Evolutionary Transitions in Individuality”, <em>Philosophy &amp;
Theory in Biology</em>, 7(1). doi:10.3998/ptb.6959004.0007.001 </li>

<li>–––, 2016, “Generalizing Contextual
Analysis”, <em>Acta Biotheoretica</em>, 64(2): 197–217.
doi:10.1007/s10441-016-9280-5 </li>

<li>Brandon, Robert N., 1990, <em>Adaptation and Environment</em>,
Princeton, NJ: Princeton University Press.</li>

<li>–––, 1996, <em>Concepts and Methods in
Evolutionary Biology</em>, Cambridge: Cambridge University Press.</li>

<li>Brookes, Bertram C., 1956, “An Introduction to the
Mathematical Theory of Information”, <em>The Mathematical
Gazette</em>, 40(333): 170–180. doi:10.2307/3608805</li>

<li>Brooks, Daniel R. and Edward O. Wiley, 1988, <em>Evolution as
Entropy: Toward a Unified Theory of Biology</em>, second edition,
Chicago: University of Chicago Press.</li>

<li>Caporael, Linnda, 2003, “Repeated Assembly”, in Steven
J. Scher &amp; Frederick Rauscher (eds.), <em>Evolutionary
Psychology</em>, New York: Springer US, pp. 71–89.
doi:10.1007/978-1-4615-0267-8_4 </li>

<li>Charbonneau, Mathieu, 2014, “Populations Without
Reproduction”, <em>Philosophy of Science</em>, 81(5):
727–740. doi:10.1086/677203 </li>

<li>Claidière, Nicolas and Jean-Baptiste André, 2012,
“The Transmission of Genes and Culture: A Questionable
Analogy”, <em>Evolutionary Biology</em>, 39(1): 12–24.
doi:10.1007/s11692-011-9141-8 </li>

<li>Crick, Francis H.C., 1958, “On Protein Synthesis”, in
Frank K. Sanders (ed.), <em>Symposia of the Society for Experimental
Biology, Number XII: The Biological Replication of
Macromolecules</em>, Cambridge; New York: Cambridge University Press,
pp. 138–163.
 [<a href="https://profiles.nlm.nih.gov/ps/retrieve/ResourceMetadata/SCBBZY" target="other">Crick 1958 available online</a>]</li>
 
<li>–––, 1970, “Central Dogma of Molecular
Biology”, <em>Nature</em>, 227(5258): 561–3.
doi:10.1038/227561a0</li>

<li>Dawkins, Richard, 1976, <em>The Selfish Gene</em>, New York:
Oxford University Press.</li>

<li>–––, 1982a, “Replicators and
Vehicles”, in <em>Current Problems in Sociobiology</em>, King's
College Sociobiology Group (eds), Cambridge: Cambridge University
Press, pp. 45–64.</li>

<li>–––, 1982b, <em>The Extended Phenotype: The Gene
as the Unit of Selection</em>, Oxford; San Francisco: Freeman.</li>

<li>–––, 1986, <em>The Blind Watchmaker</em>,
Harlow: Longman Scientific and Technical.</li>

<li>–––, 1994, “Burying the Vehicle”,
<em>Behavioral and Brain Sciences</em>, 17(4): 616–617.
doi:10.1017/S0140525X00036207 </li>

<li>Delius, Juan, 1991, “The Nature of Culture”, in Marion
Stamp Dawkins, Tim R. Halliday, &amp; Richard Dawkins (eds.), <em>The
Tinbergen Legacy</em>, London: Chapman and Hall.</li>

<li>Doolittle, W. Ford, 2014, “Natural Selection Through
Survival Alone, and the Possibility of Gaia”, <em>Biology and
Philosophy</em>, 29: 415–423. doi:10.1007/s10539-013-9384-0
</li>

<li>Earnshaw-Whyte, Eugene, 2012, “Increasingly Radical Claims
About Heredity and Fitness”, <em>Philosophy of Science</em>,
79(3): 396–412. doi:10.1086/666060 </li>

<li>Eigen, Manfred and Leo de Maeyer, 1966, “Chemical Means of
Information Storage and Readout in Biological Systems”,
<em>Naturwissenschaften</em>, 53(3): 50–57.
doi:10.1007/BF00594747 </li>

<li>Eigen, Manfred, Christof K. Biebricher, Michael Gebinoga, and
William C. Gardiner, 1991, “The Hypercycle. Coupling of RNA and
Protein Biosynthesis in the Infection Cycle of an RNA
Bacteriophage”, <em>Biochemistry</em>, 30(46):
11005–11018. doi:10.1021/bi00110a001</li>

<li>El Mouden, Claire, Jean-Baptiste André, Olivier Morin, and
Daniel Nettle, 2014, “Cultural Transmission and the Evolution of
Human Behaviour: A General Approach Based on the Price
Equation”, <em>Journal of Evolutionary Biology</em>, 27(2):
231–241. doi:10.1111/jeb.12296 </li>

<li>Fisher, Ronald Aylmer, 1930, <em>The Genetical Theory of Natural
Selection</em>, Oxford: Clarendon Press.</li>

<li>Gatlin, Lila L., 1972, <em>Information Theory and the Living
System</em>, New York: Columbia University Press.</li>

<li>Gilbert, Scott F., 1991, <em>A Conceptual History of Modern
Embryology</em>, New York: Plenum Press.</li>

<li>Godfrey-Smith, Peter, 2000a, “Information, Arbitrariness,
and Selection: Comments on Maynard Smith”, <em>Philosophy of
Science</em>, 67(2): 202–207. doi:10.1086/392770</li>

<li>–––, 2000b, “The Replicator in
Retrospect”, <em>Biology and Philosophy</em>, 15(3):
403–423. doi:10.1023/A:1006704301415</li>

<li>–––, 2007, “Conditions for Evolution by
Natural Selection”, <em>The Journal of Philosophy</em>, 104(10):
489–516. doi:10.5840/jphil2007104103</li>

<li>–––, 2009, <em>Darwinian Populations and Natural
Selection</em>, Oxford: Oxford University Press.
doi:10.1093/acprof:osobl/9780199552047.001.0001</li>

<li>Godfrey-Smith, Peter and Benjamin Kerr, 2002, “Group Fitness
and Multi-Level Selection: Replies to Commentaries”, <em>Biology
and Philosophy</em>, 17(4): 539–549.
doi:10.1023/A:1020583723772</li>

<li>Griesemer, James R., 2000a, “Development, Culture, and the
Units of Inheritance”, <em>Philosophy of Science</em>,
67(Supplement. Proceedings of the 1998 Biennial Meetings of the
Philosophy of Science Association. Part II: Symposia Papers):
S348–S368. doi:10.1086/392831</li>

<li>–––, 2000b, “The Units of Evolutionary
Transition”, <em>Selection</em>, 1(1–3): 67–80.</li>

<li>–––, 2002, “What Is ‘Epi’
About Epigenetics?” <em>Annals of the New York Academy of
Sciences</em>, 981(1): 97–110.
doi:10.1111/j.1749-6632.2002.tb04914.x </li>

<li>–––, 2005, “The Informational Gene and the
Substantial Body: On the Generalization of Evolutionary Theory by
Abstraction”, in Martin R. Jones &amp; Nancy Cartwright (eds.),
<em>Idealization XII: Correcting the Model. Idealization and
Abstraction in the Sciences</em>, Amsterdam: Rodopi Publishers, pp.
59–115.</li>

<li>–––, 2014, “Reproduction and Scaffolded
Developmental Processes: An Integrated Evolutionary
Perspective”, in Alessandro Minelli &amp; Thomas Pradeu (eds.),
<em>Towards a Theory of Development</em>, Oxford: Oxford University
Press, pp. 183–202.
doi:10.1093/acprof:oso/9780199671427.003.0012 </li>

<li>Griffiths, Paul E., 2001, “Genetic Information: A Metaphor
in Search of a Theory”, <em>Philosophy of Science</em>, 68(3):
394–412. doi:10.1086/392891 </li>

<li>Griffiths, Paul E. and Russell D. Gray, 1994a,
“Developmental Systems and Evolutionary Explanation”,
<em>Journal of Philosophy</em>, 91(6): 277–304.
doi:10.2307/2940982</li>

<li>–––, 1994b, “Replicators and Vehicles? or
Developmental Systems”, <em>Behavioral and Brain Sciences</em>,
17(4): 623–624. doi:10.1017/S0140525X00036281</li>

<li>Griffiths, Paul E. and Karola Stotz, 2007, “Gene”, in
Michael Ruse &amp; David Hull (eds.), <em>Cambridge Companion to
Philosophy of Biology</em>, Cambridge: Cambridge University Press, pp.
85–102. doi:10.1017/CCOL9780521851282.005</li>

<li>–––, 2013, <em>Genetics and Philosophy: An
Introduction</em>, Cambridge: Cambridge University Press.
doi:10.1017/CBO9780511744082</li>

<li>Haig, David, 2012, “The Strategic Gene”, <em>Biology
and Philosophy</em>, 27(4): 461–479.
doi:10.1007/s10539-012-9315-5 </li>

<li>Haldane, John Burdon Sanderson, 1929, “Origin of
Life”, <em>The Rationalist Annual</em>, 148: 3–10.</li>

<li>Hamilton, Andrew and Matthew H. Haber, 2006, “Clades Are
Reproducers”, <em>Biological Theory</em>, 1(4): 381–391.
doi:10.1162/biot.2006.1.4.381</li>

<li>Hamilton, William D., 1964, “The Genetical Evolution of
Social Behaviour. II”, <em>Journal of Theoretical Biology</em>,

7(1): 17–52. doi:10.1016/0022-5193(64)90039-6 </li>

<li>Hsu, Laura L. and Sidney W. Fox, 1976, “Interactions Between
Diverse Proteinoids and Microspheres in Simulation of Primordial
Evolution”, <em>Biosystems</em>, 8(2): 89–101.
doi:10.1016/0303-2647(76)90012-5 </li>

<li>Hull, David L., 1980, “Individuality and Selection”,
<em>Annual Review of Ecology and Systematics</em>, 11: 311–332.
doi:10.1146/annurev.es.11.110180.001523</li>

<li>–––, 1981, “Units of Evolution: A
Metaphysical Essay”, in U.L. Jensen &amp; Rom Harré

(eds.), <em>The Philosophy of Evolution</em>, Brighton UK: Harvester
Press, pp. 23–44.</li>

<li>–––, 1988, “Interactors Versus
Vehicles”, in Henry C. Plotkin (ed.), <em>The Role of Behavior
in Evolution</em>, Cambridge, MA: MIT Press, pp. 19–50.</li>

<li>Hull, David L., Rodney E. Langman, and Sigrid S. Glenn, 2001,
“A General Account of Selection: Biology, Immunology, and
Behavior”, <em>Behavioral and Brain Sciences</em>, 24(3):
511–28; discussion 528–73. doi:10.1017/S0140525X0156416X
</li>

<li>Jablonka, Eva and Marion J. Lamb, 2005, <em>Evolution in Four
Dimensions: Genetic, Epigenetic, Behavioral, and Symbolic Variation in
the History of Life</em>, Cambridge, MA: MIT Press.</li>

<li>–––, 2007, “Précis of <em>Evolution
in Four Dimensions</em>”, <em>Behavioral and Brain
Sciences</em>, 30(4): 353–365.
doi:10.1017/S0140525X07002221</li>

<li>Jablonka, Eva and Gal Raz, 2009, “Transgenerational
Epigenetic Inheritance: Prevalence, Mechanisms, and Implications for
the Study of Heredity and Evolution”, <em>Quarterly Review of
Biology</em>, 84(2): 131–76. doi:10.1086/598822</li>

<li>Jacob, François, 1973, <em>The Logic of Life: A History of
Heredity</em>, New York: Pantheon Books.</li>

<li>Jacobson, Homer, 1958, “On Models of Reproduction”,
<em>American Scientist</em>, 46(3): 255–284.</li>

<li>Johnson, Horton A., 1970, “Information Theory in Biology
After 18 Years”, <em>Science</em>, 168(3939): 1545–1550.
doi:10.1126/science.168.3939.1545 </li>

<li>Keller, Laurent, 1999, <em>Levels of Selection in Evolution</em>,
Princeton NJ: Princeton University Press.</li>

<li>Kerr, Benjamin and Peter Godfrey-Smith, 2002, “Individualist
and Multi-Level Perspectives on Selection in Structured
Populations”, <em>Biology and Philosophy</em>, 17(4):
477–517, with commentaries, and replies by the authors:
Godfrey-Smith and Kerr 2002. doi:10.1023/A:1020504900646</li>

<li>Kolb, Vera M., 2016, “Origins of Life: Chemical and
Philosophical Approaches”, <em>Evolutionary Biology</em>, 43(4):
506–515. doi:10.1007/s11692-015-9361-4 </li>

<li>Koonin, Eugene V., 2012, “Does the Central Dogma Still
Stand?” <em>Biology Direct</em>, 7: 27–27.
doi:10.1186/1745-6150-7-27 </li>

<li>–––, 2015, “Why the Central Dogma? On the
Nature of the Great Biological Exclusion Principle”, <em>Biology
Direct</em>, 10: 52. doi:10.1186/s13062-015-0084-3 </li>

<li>Lehrman, Daniel S., 1953, “A Critique of Konrad
Lorenz’s Theory of Instinctive Behavior”, <em>The
Quarterly Review of Biology</em>, 28(4): 337–363.
doi:10.1086/399858</li>

<li>Lewens, Tim, 2015, <em>Cultural Evolution: Conceptual
Challenges</em>, Oxford, UK: Oxford University Press.
doi:10.1093/acprof:oso/9780199674183.001.0001</li>

<li>Lewontin, Richard C., 1970, “The Units of Selection”,
<em>Annual Review of Ecology and Systematics</em>, 1: 1–18.
doi:10.1146/annurev.es.01.110170.000245</li>

<li>–––, 1991, <em>Biology as Ideology: The Doctrine
of DNA</em>, Toronto: Anansi Press.</li>

<li>Linschitz, H., 1953, <em>Information Theory in Biology</em>,
Urbana, IL: University of Illinois Press.</li>

<li>Lloyd, Elisabeth Anne, 1988, <em>The Structure and Confirmation of
Evolutionary Theory</em>, (Contributions in Philosophy, no. 37), New
York: Greenwood Press. </li>

<li>–––, 1992, “Unit of Selection”, in
<em>Keywords In Evolutionary Biology</em>, Evelyn Fox Keller and
Elisabeth A. Lloyd (eds), Cambridge MA: Harvard University Press, pp.
334–342.</li>

<li>Lu, Qiaoying and Pierrick Bourrat, 2018, “The Evolutionary
Gene and the Extended Evolutionary Synthesis”, <em>The British
Journal for the Philosophy of Science</em>, 69(5):775-800. doi:10.1093/bjps/axx019
</li>

<li>Maclaurin, James, 1998, “Reinventing Molecular Weismannism:
Information in Evolution”, <em>Biology and Philosophy</em>,
13(1): 37–59. doi:10.1023/A:1006573021270</li>

<li>Margalef, Ramón, 1957 [1958], “La teoría de la
información en ecología”, <em>Memorias de la Real
Academia de Ciencias y Artes de Barcelona</em>, 23: 373–449.
Translated as “Information Theory in Ecology”, by Wendell
Hall in <em>Yearbook of the Society for General Systems Research</em>,
3: 36–71.
 [<a href="http://www.vliz.be/en/imis?module=ref&amp;refid=126440" target="other">Margalef 1957 [1958] available online (English)</a>]
 </li>

<li>Maynard Smith, John, 2000, “The Concept of Information in
Biology”, <em>Philosophy of Science</em>, 67(2): 177–194.
doi:10.1086/392768</li>

<li>Maynard Smith, John and Eörs Szathmáry, 1995, <em>The
Major Transitions in Evolution</em>, Oxford, New York and Heidelberg:
WH Freeman/Spektrum.</li>

<li>Mitchell, Sandra D., 2009, <em>Unsimple Truths: Science,
Complexity, and Policy</em>, Chicago: University of Chicago Press.
</li>

<li>Neumann, John Von, 1966, <em>Theory of Self-Reproducing
Automata</em>, Arthur W. Burks (ed.), Urbana IL: University of
Illinois Press.</li>

<li>Okasha, Samir, 2006, <em>Evolution and the Levels of
Selection</em>, Oxford: Clarendon.
doi:10.1093/acprof:oso/9780199267972.001.0001</li>

<li>–––, 2016, “The Relation Between Kin and
Multilevel Selection: An Approach Using Causal Graphs”, <em>The
British Journal for the Philosophy of Science</em>, 67(2):
435–470. doi:10.1093/bjps/axu047 </li>

<li>Oparin, Aleksandr Ivanovich, 1936 [1938], <em>Vozniknovenie zhizni
na zemle</em>, Moscow: Izd. Akad. Nauk SSSR. Translated as <em>The
Origin of Life</em>, Sergius Morgulis (trans.), New York: Macmillan.
</li>

<li>Oyama, Susan, 1985, <em>The Ontogeny of Information: Developmental
Systems and Evolution</em>, Cambridge; New York: Cambridge University
Press. </li>

<li>–––, 2000, <em>The Ontogeny of Information:
Developmental Systems and Evolution</em>, second edition, Durham, NC:
Duke University Press. </li>

<li>Oyama, Susan, Paul E. Griffiths, &amp; Russell D. Gray (eds.),
2001, <em>Cycles of Contingency: Developmental Systems and
Evolution</em>, Cambridge MA; London: MIT Press: A Bradford Book.</li>

<li>Peluffo, Alexandre E., 2015, “The ‘Genetic
Program’: Behind the Genesis of an Influential Metaphor”,
<em>Genetics</em>, 200(3): 685–696.
doi:10.1534/genetics.115.178418 </li>

<li>Pigliucci, Massimo and Gerd B. Müller, 2010, <em>Evolution:
The Extended Synthesis</em>, Cambridge MA: MIT Press.</li>

<li>Quastler, Henry, 1953, <em>Information Theory in Biology</em>,
Urbana IL: University of Illinois Press.</li>

<li>Rashevsky, N., 1955, “Life, Information Theory, and
Topology”, <em>The Bulletin of Mathematical Biophysics</em>,
17(3): 229–235. doi:10.1007/BF02477860 </li>

<li>Richerson, Peter J. and Robert Boyd, 2008, <em>Not by Genes Alone:
How Culture Transformed Human Evolution</em>, Chicago: University of
Chicago Press.</li>

<li>Sarkar, Sahotra, 1998, <em>Genetics and Reductionism</em>,
Cambridge; New York: Cambridge University Press.
doi:10.1017/CBO9781139173216</li>

<li>–––, 2000, “Information in Genetics and
Developmental Biology: Comments on Maynard Smith”,
<em>Philosophy of Science</em>, 67(2): 208–213.
doi:10.1086/392771</li>

<li>Schrödinger, Erwin, 1944, <em>What Is Life? The Physical
Aspect of the Living Cell</em>, Cambridge: Cambridge University
Press.</li>

<li>Shannon, Claude E., 1948, “A Mathematical Theory of
Communication”, <em>The Bell System Technical Journal</em>,
27(3): 379–423, 27(4): 623–656.
doi:10.1002/j.1538-7305.1948.tb01338.x and
doi:10.1002/j.1538-7305.1948.tb00917.x</li>

<li>Sober, Elliott and David Sloan Wilson, 1998, <em>Unto Others: The
Evolution and Psychology of Unselfish Behavior</em>, Cambridge, MA:
Harvard University Press.</li>

<li>Stent, Gunther S., 1977, “You Can Take the Ethics Out of
Altruism but You Can’t Take the Altruism Out of Ethics”,
<em>Hastings Center Report</em>, 7(6): 33–36.
doi:10.2307/3560881</li>

<li>Sterelny, Kim, 1996, “The Return of the Group”,
<em>Philosophy of Science</em>, 63(4): 562–584.
doi:10.1086/289977</li>

<li>–––, 2000a, “Development, Evolution, and
Adaptation”, <em>Philosophy of Science</em>, 67(Supplement.
Proceedings of the 1998 Biennial Meetings of the Philosophy of Science
Association): S369–S387. doi:10.1086/392832</li>

<li>–––, 2000b, “The ‘Genetic
Program’ Program: A Commentary on Maynard Smith on Information
in Biology”, <em>Philosophy of Science</em>, 67(2):
195–201. doi:10.1086/392769</li>

<li>–––, 2001, “Niche Construction,
Developmental Systems, and the Extended Replicator”, in Susan
Oyama et al. 2001: 333–350.</li>

<li>–––, 2006, “Memes Revisited”,
<em>British Journal for the Philosophy of Science</em>, 57(1):
145–165. doi:10.1093/bjps/axi157 </li>

<li>Sterelny, Kim, Kelly C. Smith, and Michael Dickison, 1996,
“The Extended Replicator”, <em>Biology and
Philosophy</em>, 11(3): 377–403. doi:10.1007/BF00128788 </li>

<li>Sternberg, Richard, 2008, “DNA Codes and Information: Formal
Structures and Relational Causes”, <em>Acta Biotheoretica</em>,
56(3): 205–232. doi:10.1007/s10441-008-9049-6 </li>

<li>Szathmáry, Eörs and John Maynard Smith, 1997,
“From Replicators to Reproducers: The First Major Transitions
Leading to Life”, <em>Journal of Theoretical Biology</em>,
187(4): 555–571. doi:10.1006/jtbi.1996.0389 </li>

<li>Šustar, Predrag, 2007, “Crick’s Notion of
Genetic Information and the ’Central Dogma’ of Molecular
Biology”, <em>British Journal for the Philosophy of
Science</em>, 58(1): 13–24. doi:10.1093/bjps/axl018 </li>

<li>Thieffry, Denis and Sahotra Sarkar, 1998, “Forty Years Under
the Central Dogma”, <em>Trends in Biochemical Sciences</em>,
23(8): 312–316. doi:10.1016/S0968-0004(98)01244-4 </li>

<li>Wallace, Rodrick and Robert G. Wallace, 1998, “Information
Theory, Scaling Laws and the Thermodynamics of Evolution”,
<em>Journal of Theoretical Biology</em>, 192: 545–559.</li>

<li>Watson, James D. and Francis H.C. Crick, 1953, “Molecular
Structure of Nucleic Acids: A Structure for Deoxyribose Nucleic
Acid”, <em>Nature</em>, 171(4356): 737–738.
doi:10.1038/171737a0 </li>

<li>Wiener, Norbert, 1948, <em>Cybernetics, or, Control and
Communication in the Animal and the Machine</em>, Cambridge MA:
Technology Press.</li>

<li>Wilkins, John S., 1998, “What’s in a Meme? Reflections
from the Perspective of the History and Philosophy of Evolutionary
Biology”, <em>Journal of Memetics – Evolutionary Models of
Information Transmission</em>, 2(1): 2–33.
 [<a href="http://cfpm.org/jom-emit/1998/vol2/wilkins_js.html" target="other">Wilkins 1998 available online</a>]
 <!-- not sure of the reliability of the above link since the journal is no
 longer active hence keeping the below also 
 [<a href="https://www.dropbox.com/s/v6m3p3auavncjtv/WIAM.pdf" target="other">Wilkins 1998 draft available online</a>]
 --> 
</li>

<li>Wilkins, John S., Clem Stanyon, and Ian Musgrave, 2012,
“Selection Without Replicators: The Origin of Genes, and the
Replicator/Interactor Distinction in Etiobiology”, <em>Biology
and Philosophy</em>, 27(2): 215–239.
doi:10.1007/s10539-011-9298-7 </li>

<li>Williams, George C., 1966, <em>Adaptation and Natural Selection: A
Critique of Some Current Evolutionary Thought</em>, Princeton NJ:
Princeton University Press.</li>

<li>Wimsatt, William C. and James R. Griesemer, 2007,
“Reproducing Entrenchments to Scaffold Culture: The Central Role
of Development in Cultural Evolution”, <em>Integrating Evolution
and Development: From Theory to Practice</em>, Roger Sansom and Robert
N. Brandon (eds), Cambridge, MA: MIT Press, pp. 227–324.</li>

<li>Winnie, John A., 2000, “Information and Structure in
Molecular Biology: Comments on Maynard Smith”, <em>Philosophy of
Science</em>, 67(3): 517–526. doi:10.1086/392793</li>
</ul>

</div> 
<div id="academic-tools">

<h2 id="Aca">Academic Tools</h2>

<blockquote>
<table class="vert-top">
<tbody><tr>
<td><img src="../../symbols/sepman-icon.jpg" alt="sep man icon" /></td>
<td><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/archinfo.cgi?entry=replication&amp;archive=win2019" target="other">How to cite this entry</a>.</td>
</tr>

<tr>
<td><img src="../../symbols/sepman-icon.jpg" alt="sep man icon" /></td>
<td><a href="https://leibniz.stanford.edu/friends/preview/replication/" target="other">Preview the PDF version of this entry</a> at the
 <a href="https://leibniz.stanford.edu/friends/" target="other">Friends of the SEP Society</a>.</td>
</tr>

<tr>
<td><img src="../../symbols/inpho.png" alt="inpho icon" /></td>
<td><a href="https://www.inphoproject.org/entity?sep=replication&amp;redirect=True" target="other">Look up this entry topic</a>
 at the <a href="https://www.inphoproject.org/" target="other">Indiana Philosophy Ontology Project</a>
 (InPhO).</td>
</tr>

<tr>
<td><img src="../../symbols/pp.gif" alt="phil papers icon" /></td>
<td><a href="http://philpapers.org/sep/replication/" target="other">Enhanced bibliography for this entry</a>
at <a href="http://philpapers.org/" target="other">PhilPapers</a>, with links to its database.</td>
</tr>

</tbody></table>
</blockquote>

</div>

<div id="other-internet-resources">

<h2 id="Oth">Other Internet Resources</h2>

<ul>

 <li><a href="https://embryo.asu.edu/pages/john-von-neumanns-cellular-automata" target="other">Information on von Neumann’s self-reproducing automata</a>.</li>
 
 <li><a href="https://www.atdbio.com/content/14/Transcription-Translation-and-Replication" target="other">Introduction to the molecular aspects of DNA replication</a></li>
 
 <li><a href="https://books.google.com/ngrams/graph?content=replication&amp;year_start=1800&amp;year_end=2000&amp;corpus=15&amp;smoothing=3&amp;share=&amp;direct_url=t1%3B%2Creplication%3B%2Cc0" target="other">Google Ngram Viewer for replication</a>
 (see
 <a href="#fig1">Figure 1 above</a>,
 note there are some differences from the figure due to changes in the
database Ngram uses)</li>
</ul>
</div>

<div id="related-entries">

<h2 id="Rel">Related Entries</h2>

<p>

 <a href="../biology-philosophy/">biology: philosophy of</a> |
 <a href="../epigenesis/">developmental biology: epigenesis and preformationism</a> |
 <a href="../evolution/">evolution</a> |
 <a href="../evolution-cultural/">evolution: cultural</a> |
 <a href="../fitness/">fitness</a> |
 <a href="../gene/">gene</a> |
 <a href="../genotype-phenotype/">genetics: genotype/phenotype distinction</a> |
 <a href="../heredity/">heritability</a> |
 <a href="../information-biological/">information: biological</a> |
 <a href="../inheritance-systems/">inheritance systems</a> |
 <a href="../life/">life</a> |
 <a href="../molecular-biology/">molecular biology</a> |
 <a href="../natural-selection/">natural selection</a> |
 <a href="../selection-units/">natural selection: units and levels of</a>

</p>

</div>

<div id="acknowledgments">

<h3>Acknowledgments</h3>

<p>
The original version of this article, entitled
“Replication” was written by the late David Hull. In
January 2005, John Wilkins became a co-author, with the responsibility
for revising and updating the entry, so as to keep it current. Thus,
revisions on or after January 2005 were introduced by Wilkins, and
from October 2018, by Wilkins and Pierrick Bourrat. Although the
current version includes some of Hull’s original essay, it does
not represent his final views.</p>

<p>
The authors would like to thank Peter Godfrey-Smith and James
Griesemer for their suggestions and comments, and the reviewers who
made many useful comments and suggestions.</p>

</div>

<script type="text/javascript" src="local.js"></script>
<script type="text/javascript" src="../../MathJax/MathJax.js?config=TeX-MML-AM_CHTML"></script>

</div><!-- #aueditable --><!--DO NOT MODIFY THIS LINE AND BELOW-->

<!-- END ARTICLE HTML -->

</div> <!-- End article-content -->

  <div id="article-copyright">
    <p>
 <a href="../../info.html#c">Copyright © 2018</a> by

<br />
<a href="https://philpeople.org/profiles/john-wilkins-1" target="other">John S. Wilkins</a>
&lt;<a href="mailto:john%40wilkins%2eid%2eau"><em>john<abbr title=" at ">@</abbr>wilkins<abbr title=" dot ">.</abbr>id<abbr title=" dot ">.</abbr>au</em></a>&gt;<br />
Pierrick Bourrat
&lt;<a href="mailto:p%2ebourrat%40gmail%2ecom"><em>p<abbr title=" dot ">.</abbr>bourrat<abbr title=" at ">@</abbr>gmail<abbr title=" dot ">.</abbr>com</em></a>&gt;
    </p>
  </div>

</div> <!-- End article -->

<!-- NOTE: article banner is outside of the id="article" div. -->
<div id="article-banner" class="scroll-block">
     <div id="article-banner-content">
  <a href="../../fundraising/">
  Open access to the SEP is made possible by a world-wide funding initiative.<br />
  Please Read How You Can Help Keep the Encyclopedia Free</a>
 </div>


</div> <!-- End article-banner -->

    </div> <!-- End content -->

    <div id="footer">

      <div id="footer-menu">
        <div class="menu-block">
          <h4><i class="icon-book"></i> Browse</h4>
          <ul role="menu">
            <li><a href="../../contents.html">Table of Contents</a></li>
            <li><a href="../../new.html">New in this Archive</a></li>
            
            <li><a href="../../published.html">Chronological</a></li>
            <li><a href="../../../../archives/">Archives <i class="icon-external-link"></i></a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-info-sign"></i> About</h4>
          <ul role="menu">
            <li><a href="../../info.html">Editorial Information</a></li>
            <li><a href="../../about.html">About the SEP</a></li>
            <li><a href="../../board.html">Editorial Board</a></li>
            <li><a href="../../cite.html">How to Cite the SEP</a></li>
            <li><a href="../../special-characters.html">Special Characters</a></li>
            
            <li><a href="../../../../contact.html">Contact <i class="icon-external-link"></i></a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-leaf"></i> Support SEP</h4>
          <ul role="menu">
            <li><a href="../../../../support/">Support the SEP</a></li>
            <li><a href="../../../../support/friends.html">PDFs for SEP Friends</a></li>
            <li><a href="../../../../support/donate.html">Make a Donation</a></li>
            <li><a href="../../../../support/sepia.html">SEPIA for Libraries</a></li>
          </ul>
        </div>
      </div> <!-- End footer menu -->

      <div id="mirrors">
        <div id="mirror-info">
          <h4><i class="icon-globe"></i> Mirror Sites</h4>
          <p>View this site from another server:</p>
        </div>
                <div class="btn-group">
<a class="btn dropdown-toggle" data-toggle="dropdown" href="https://plato.stanford.edu/"><span class="flag flag-usa"></span> USA (Main Site) <span class="caret"></span><span class="mirror-source">CSLI, Stanford University</span></a>          <ul class="dropdown-menu">
            <li><a href="https://stanford.library.sydney.edu.au/archives/win2019/entries/replication/"><span class="flag flag-australia"></span> Australia <span class="mirror-source">Library, University of Sydney</span></a>           </li>
            <li><a href="https://seop.illc.uva.nl/archives/win2019/entries/replication/"><span class="flag flag-netherlands"></span> Netherlands <span class="mirror-source">ILLC, University of Amsterdam</span></a>           </li>
          </ul>
        </div>
      </div> <!-- End mirrors -->
      
      <div id="site-credits">
        <p class="csli-logo"><a href="https://www-csli.stanford.edu/"><img src="../../symbols/SU_csli.png" width="355" alt="Stanford Center for the Study of Language and Information" /></a></p>
        <p>The Stanford Encyclopedia of Philosophy is <a href="../../info.html#c">copyright © 2016</a> by <a href="http://mally.stanford.edu/">The Metaphysics Research Lab</a>, Center for the Study of Language and Information (CSLI), Stanford University</p>
        <p>Library of Congress Catalog Data: ISSN 1095-5054</p>
      </div> <!-- End site credits -->

    </div> <!-- End footer -->

  </div> <!-- End container -->

   <!-- NOTE: Script required for drop-down button to work (mirrors). -->
  <script>
    $('.dropdown-toggle').dropdown();
  </script>





</body></html>