<!DOCTYPE html><!--[if lt IE 7]> <html class="ie6 ie"> <![endif]--><!--[if IE 7]>    <html class="ie7 ie"> <![endif]--><!--[if IE 8]>    <html class="ie8 ie"> <![endif]--><!--[if IE 9]>    <html class="ie9 ie"> <![endif]--><!--[if !IE]> --><html xmlns="http://www.w3.org/1999/xhtml"><!-- <![endif]--><head>
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<title>
Evolutionary Genetics (Stanford Encyclopedia of Philosophy/Winter 2019 Edition)
</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="robots" content="noarchive, noodp" />
<meta property="citation_title" content="Evolutionary Genetics" />
<meta property="citation_author" content="Wade, Michael" />
<meta property="citation_publication_date" content="2005/01/14" />
<meta name="DC.title" content="Evolutionary Genetics" />
<meta name="DC.creator" content="Wade, Michael" />
<meta name="DCTERMS.issued" content="2005-01-14" />
<meta name="DCTERMS.modified" content="2005-01-14" />

<!-- NOTE: Import webfonts using this link: -->
<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,300,600,200&amp;subset=latin,latin-ext" rel="stylesheet" type="text/css" />

<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap.min.css" />
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/bootstrap-responsive.min.css" />
<link rel="stylesheet" type="text/css" href="../../css/font-awesome.min.css" />
<!--[if IE 7]> <link rel="stylesheet" type="text/css" href="../../css/font-awesome-ie7.min.css"> <![endif]-->
<link rel="stylesheet" type="text/css" media="screen,handheld" href="../../css/style.css" />
<link rel="stylesheet" type="text/css" media="print" href="../../css/print.css" />
<link rel="stylesheet" type="text/css" href="../../css/entry.css" />
<!--[if IE]> <link rel="stylesheet" type="text/css" href="../../css/ie.css" /> <![endif]-->
<script type="text/javascript" src="../../js/jquery-1.9.1.min.js"></script>
<script type="text/javascript" src="../../js/bootstrap.min.js"></script>

<!-- NOTE: Javascript for sticky behavior needed on article and ToC pages -->
<script type="text/javascript" src="../../js/jquery-scrolltofixed-min.js"></script>
<script type="text/javascript" src="../../js/entry.js"></script>

<!-- SEP custom script -->
<script type="text/javascript" src="../../js/sep.js"></script>
</head>

<!-- NOTE: The nojs class is removed from the page if javascript is enabled. Otherwise, it drives the display when there is no javascript. -->
<body class="archive article" id="pagetopright">
<div id="container">
<div id="header-wrapper">
  <div id="header">
    <div id="branding">
      <div id="site-logo"><a href="../../index.html"><img src="../../symbols/sep-man-red.png" alt="SEP logo" /></a></div>
      <div id="site-title"><a href="../../index.html">Stanford Encyclopedia of Philosophy Archive<div id="site-subtitle">Winter 2019 Edition</div></a></div>
    </div>
    <div id="navigation">
      <div class="navbar">
        <div class="navbar-inner">
          <div class="container">
            <button class="btn btn-navbar collapsed" data-target=".collapse-main-menu" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Menu </button>
            <div class="nav-collapse collapse-main-menu collapse">
              <ul class="nav">
                <li class="dropdown"><a id="drop1" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-book"></i> Browse</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop1">
                    <li><a href="../../contents.html">Table of Contents</a></li>
                    <li><a href="../../new.html">New in this Archive</a></li>
                    
                    <li><a href="../../published.html">Chronological</a></li>
                    <li><a href="../../../../archives/">Archives <i class="icon-external-link"></i></a></li>
                  </ul>
                </li>
                <li class="dropdown"><a id="drop2" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-info-sign"></i> About</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop2">
                    <li><a href="../../info.html">Editorial Information</a></li>
                    <li><a href="../../about.html">About the SEP</a></li>
                    <li><a href="../../board.html">Editorial Board</a></li>
                    <li><a href="../../cite.html">How to Cite the SEP</a></li>
                    <li><a href="../../special-characters.html">Special Characters</a></li>
                    
                    <li><a href="../../../../contact.html">Contact <i class="icon-external-link"></i></a></li>
                  </ul>
                </li>
                <li class="dropdown"><a id="drop3" href="#" class="dropdown-toggle" data-toggle="dropdown" role="button"><i class="icon-leaf"></i> Support SEP</a>
                  <ul class="dropdown-menu" role="menu" aria-labelledby="drop3">
                    <li><a href="../../../../support/">Support the SEP</a></li>
                    <li><a href="../../../../support/friends.html">PDFs for SEP Friends</a></li>
                    <li><a href="../../../../support/donate.html">Make a Donation</a></li>
                    <li><a href="../../../../support/sepia.html">SEPIA for Libraries</a></li>
                  </ul>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
    <!-- End navigation -->
    
    <div id="search">
      <form id="search-form" method="get" action="../../../../search/searcher.py">
        <input type="search" name="query" placeholder="Search this archive" />
<input type="hidden" name="archive" value="win2019" />

        <div class="search-btn-wrapper"><button class="btn search-btn" type="submit"><i class="icon-search"></i></button></div>
      </form>
    </div>
    <!-- End search --> 
    
  </div>
  <!-- End header --> 
</div>
<!-- End header wrapper -->

<div id="content">

<!-- Begin article sidebar -->
<div id="article-sidebar" class="sticky" style="z-index: 999;">
  <div class="navbar">
    <div class="navbar-inner">
      <div class="container">
        <button class="btn btn-navbar collapsed" data-target=".collapse-sidebar" data-toggle="collapse" type="button"> <i class="icon-reorder"></i> Entry Navigation </button>
        <div id="article-nav" class="nav-collapse collapse-sidebar collapse">
          <ul class="nav">
            <li><a href="#toc">Entry Contents</a></li>
            <li><a href="#Bib">Bibliography</a></li>
            <li><a href="#Aca">Academic Tools</a></li>
            <li><a href="https://leibniz.stanford.edu/friends/preview/evolutionary-genetics/">Friends PDF Preview <i class="icon-external-link"></i></a></li>
            <li><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/archinfo.cgi?entry=evolutionary-genetics&amp;archive=win2019">Author and Citation Info <i class="icon-external-link"></i></a> </li>
            <li><a href="#pagetopright" class="back-to-top">Back to Top <i class="icon-angle-up icon2x"></i></a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div><div></div>
<!-- End article sidebar --> 

<!-- NOTE: Article content must have two wrapper divs: id="article" and id="article-content" -->
<div id="article">
<div id="article-content">

<!-- BEGIN ARTICLE HTML -->


<div id="aueditable"><!--DO NOT MODIFY THIS LINE AND ABOVE-->

<h1>Evolutionary Genetics</h1><div id="pubinfo"><em>First published Fri Jan 14, 2005</em></div>

<div id="preamble">

<p>

<em>Evolutionary genetics</em> is the broad field of studies that
resulted from the integration of genetics and Darwinian evolution,
called the ‘modern synthesis’ (Huxley 1942), achieved
through the theoretical works of R. A. Fisher, S. Wright, and J. B. S.
Haldane and the conceptual works and influential writings of  J.
Huxley, T. Dobzhansky, and H.J. Muller. This field attempts to account
for evolution in terms of changes in gene and genotype frequencies
within populations and the processes that convert the variation with
populations into more or less permanent variation between species. In
this view, four evolutionary forces (mutation, random genetic drift,
natural selection, and gene flow) acting within and among populations
cause micro-evolutionary change and these processes are sufficient to
account for macro-evolutionary patterns, which arise in the longer
term from the collective action of these forces. That is, given very
long periods of time, the micro-evolutionary forces will eventually
give rise to the macro-evolutionary patterns that characterize the
higher taxonomic groups. Thus, the central challenge of Evolutionary
Genetics is to describe how the evolutionary forces shape the patterns
of biodiversity observed in nature.
</p>

<p>

The force of mutation is the ultimate source of new genetic variation
within populations. Although most mutations are neutral with no effect
on fitness or harmful, some mutations have a small, positive effect on
fitness and these variants are the raw materials for gradualistic
adaptive evolution. Within finite populations, random genetic drift
and natural selection affect the mutational variation.  Natural
selection is the only evolutionary force which can produce adaptation,
the fit between organism and environment, or conserve genetic states
over very long periods of time in the face of the dispersive forces of
mutation and drift. The force of migration or gene flow has effects on
genetic variation that are the opposite of those caused by random
genetic drift. Migration limits the genetic divergence of populations
and so impedes the process of speciation. The effect of each of these
evolutionary forces on genetic variation within and among populations
has been developed in great detail in the mathematical theory of
population genetics founded on the seminal works of Fisher, Wright,
and Haldane.</p>

<p>

Among the evolutionary forces, natural selection has long been
privileged in evolutionary studies because of its crucial role in
adaptation. <em>Ecological genetics</em> is the study of
evolutionary processes, especially adaptation by natural selection, in
an ecological context in order to account for phenotypic patterns
observed in nature. Where population genetics tends toward a branch of
applied mathematics founded on Mendelian axioms, often with minimal
contact with data, ecological genetics is grounded in the reciprocal
interaction between mathematical theory and empirical observations from
field and laboratory.</p>

</div>

<div id="toc">
<!--Entry Contents-->
<ul>
<li><a href="#1">1. Introduction</a></li>

<li><a href="#2">2. Classical Ecological Genetics and Polymorphism</a></li>

<li><a href="#3">3. Classical Ecological Genetics, Population Size, and Natural Selection</a></li>

<li><a href="#4">4. The Sewall Wright Effect</a></li>

<li><a href="#5">5. Interactions and their Effect on the Threshold between Natural Selection and Random Drift</a></li>

<li><a href="#6">6. Allozyme Variation and the Drift vs Selection Controversy</a></li>

<li><a href="#7">7. Sequence Variation and the Drift vs Selection Controversy</a></li>

<li><a href="#Bib">Bibliography</a></li>

<li><a href="#Aca">Academic Tools</a></li>
<li><a href="#Oth">Other Internet Resources</a></li>

<li><a href="#Rel">Related Entries</a></li>
</ul>
<!--Entry Contents-->

<hr />

</div>

<div id="main-text">

<h2><a name="1">1. Introduction</a></h2>

<p>

In this entry, I will review the history of evolutionary and
ecological genetic of research, with the emphasis on the latter. Most
investigations have focused on two of the most prominent patterns in
nature: (1) adaptation, or the ‘fit’ between organism and
environment; or, (2) polymorphism, the maintenance of two or more
phenotypic or genetic forms in a single population. The earliest
studies attempted to document the action of natural selection in wild
populations in support of Darwin. While natural selection is the only
evolutionary force that can account for adaptation, several
evolutionary forces, acting alone or in combination, can sustain a
polymorphism, at least transiently. Thus, assigning causal agency is a
much more difficult problem for explanations of polymorphism than it is
for adaptation. Conspicuous phenotypic polymorphisms, such as the
spotting patterns on butterfly wings or banding patterns of snail
shells, were the material of the earliest investigations of natural
populations. In these studies, natural selection was
‘privileged’ among the four evolutionary that change gene
frequencies as an explanation for the maintenance of
polymorphisms. I will show that the continuing emphasis on
detecting natural selection is, at least in part, historical with its
roots in the works of its founder, E. B. Ford, and his collaborators,
notably R. A. Fisher (cf., Ford 1975). In the beginning period
(1928-1950), much of the problem of assigning causal agency to the
maintenance of genetic polymorphism was resolved by definition rather
than by empirical observation (see below:
 <a href="#2">Classical Ecological Genetics and Polymorphism</a>).
 In the later period (1966-present), molecular ecological genetics
attempts to investigate a less biased sample of genetic polymorphisms,
such as allozymes and single nucleotide polymorphisms, but still
retains the early emphasis on natural selection as the single most
important evolutionary force shaping the hereditary material.</p>

<p>

Ecological genetics began at a time when the major theoretical aspects
of the Modern Synthesis were in place, when the marvels of adaptation
were clear, but when few empirical examples of natural selection in
action were available. Adaptive perfection by Fisherian gradualism
requires long periods of time wherein “… a very slight
selective effect acting for a correspondingly long time will be
equivalent to a much greater effect acting for a proportionately
shorter time” (R. A. Fisher 1921, in correspondence with
S. Wright, quoted in Provine [1986], p. 247). Very weak natural
selection, however, is an impediment to the goal of ecological
genetics to illuminate natural selection in action. Thus, the shift in
focus to understanding the role of strong natural selection in
maintaining genetic polymorphism is understandable. As put by its
founder, E. B. Ford (1975, p.3), “It [ecological genetics]
supplies the means, and the only direct means, of investigating the
actual process of evolution taking place in the present
time.”</p>

<p>

The focus of traditional ecological genetic research on the current
action of natural selection has been broadened in several ways over
the past twenty-five years. First, whereas the early studies tended to
focus on evolution in single populations, there is now a significant
emphasis in ecological genetics on the population genetic structure of
metapopulations and the roles of migration, extinction, and
colonization on evolutionary and adaptive processes. Secondly, whereas
the earliest studies emphasized chromosomes and their influence, the
advent of biochemical genetics in the late 1960s significantly
broadened the phenotype, beginning with the application of
electrophoretic methods to population studies. These studies revealed
abundant ‘hidden polymorphism’ in the new, biochemical
phenotype of enzyme mobility. These methods extended the domain of
ecological genetics from the classic ‘conspicuous phenotypic
polymorphisms’ in color, shape and behavior to the physiological
domain of enzyme function. The new emphasis on biochemical phenotype,
however, did not change the explanatory or causal framework of the
field. Determining the role of natural selection in maintaining enzyme
polymorphisms, such as the fast/slow polymorphisms of alcohol
dehydrogenase (which detoxifies environmental alcohol), superoxide
dismutase (which catalyzes the removal of free oxygen radicals), or the
esterases (which are involved in the detoxification of pesticides by
many insects), became a primary focus of investigation with the goal
of finding a selective basis for the enzyme variants in terms of
differences in their physical and kinetic properties.  Indeed, the
roots of controversy between the selectionist and neutralist schools
over the maintenance of ‘balanced’ polymorphisms
(cf. Lewontin 1974) lie in the controversy over random genetic drift
versus natural selection in early ecological genetic research (see
below). Thirdly, the more recent advent of DNA sequencing initiated
the growth of molecular phylogenetics and added not only a new
phenotype, but also a more pronounced historical dimension to
ecological genetic research.  Molecular phylogenics and comparative
sequence analysis have become the primary modern tools for the
investigation of the evolutionary patterns and processes that shape
DNA sequences. These methods have strengthened inferences regarding
biogeography, speciation, and adaptation, especially in regard to the
diversification of taxonomic lineages that attends ecological release
and adaptive radiations. They have shifted the focus from polymorphism
within species to diversification among clades and permitted the
investigation of the history of individual genes. Two new patterns in
particular have been recognized by these DNA-based methods. The first
is the preponderance of ‘purifying selection’, wherein the
conservative power of natural selection is seen as a barrier to
diversity. It is this conservative aspect of natural selection acting
at the molecular level that lends power to the investigation of the
genetic architecture of model organisms vis a vis human genetics. The
second pattern is the discovery of the existence of <em>ancient
polymorphisms</em>, molecular genetic variation whose duration may be
greater than that of the species or taxon in which it was
discovered. Natural selection, however, still remains the privileged
explanatory force in modern sequence studies.  Indeed, the search for
and documentation of uniquely molecular patterns, such as codon bias
and selective sweeps, has, if anything, elevated the focal explanatory
power of natural selection in evolutionary studies.</p>

<p>

In this entry, I will first review classical ecological genetics and
then discuss the novel kinds of processes and explanations that
accompanied the expansion of the field from single populations to
genetically structured metapopulations and from phenotypic to
biochemical and DNA sequence polymorphisms. I will show that the
central early controversy over the roles of random genetic drift and
natural selection in evolution has continued to this day, not
withstanding the apparent technological refinements afforded by the
availability of biochemical and DNA sequence data. That is, finer scale
or more reductionistic genetic data has not yet led to a resolution of
the original conceptual issues that lie at the foundation of ecological
genetics.</p>

<h2><a name="2">2. Classical Ecological Genetics and Polymorphism</a></h2>

<p>

Historically, the starting point of ecological genetic research has
been the discovery of variation within a natural population, i.e., a
phenotypic polymorphism. The subsequent goal is three-fold: (1)
determination of whether or not the polymorphism has a genetic
component; (2) determination of the frequency of each of the
polymorphic types; and, (3) determination of how natural selection
maintains the polymorphism, either alone or in combination with other
evolutionary forces. Ford (1975 p. 109; and see also Ford 1940) defines
<em>genetic polymorphism</em> as “…the occurrence
together in the same locality of two or more discontinuous forms of a
species in such proportions that the rarest of them cannot be
maintained merely by recurrent mutation”. Although recurrent
mutation in conjunction with mutation can maintain a polymorphism
indefinitely at mutation-selection balance, here Ford is clearly
interested in a more active role for natural selection in the
maintenance of polymorphism. The first task was facilitated by early
developments in population genetic theory, particularly the findings of
Fisher (1930), which Ford interpreted to mean that naturally occurring,
discontinuous phenotypic variation is “nearly always
genetic”. The reasoning stems from the theoretical findings that,
in large populations, it is unlikely that the positive and negative
effects of an allele (or chromosomal inversion) on fitness will be
exactly balanced and that the number of individuals with a rare neutral
mutation is proportional to the number of generations since its origin.
Furthermore, if truly neutral, such alleles would spread so slowly
through a large population by random genetic drift that the
‘delicate equipoise required for their neutrality will have been
upset by changes in the environment and in the genetic outfit of the
organism’ (Ford 1975, p. 110) before a neutral allele reached
appreciable frequency. In addition, recurrent mutation as a cause of
persistent polymorphism was considered most unlikely and, in fact, this
evolutionary cause is explicitly excluded from the definition of
genetic polymorphism by Ford (see above). Hence, neutral genetic
polymorphism was considered an exceptionally rare event by the founders
of ecological genetics and, consequently, such polymorphisms were the
hallmark of strong, active natural selection.</p>

<p>

Ford (1940) further distinguished two types of selective
polymorphism, transient polymorphism and balanced
polymorphism.<em>Transient polymorphism</em>, caused by a new
favorable mutation in the process of displacing its ancestral allele,
was considered unlikely, because “…advantageous genes will
usually have been already incorporated into the genetic constitution of
the species” (Ford 1975, p. 110). This and statements like it
reflect the viewpoint that organisms in nature are exquisitely adapted
to their environments by the long-acting process of Fisherian
gradualism. It is a prelude to the more explicitly adaptationist views
found in the current behavioral literature (see review in Shuster and
Wade 2003). This view of the evolutionary process as primarily one of
refinement of existing organismal adaptation is an essential part of
the Fisherian theory of evolutionary genetics (Wade and Goodnight
1998).</p>

<p>

The presumptions of a genetic basis for discontinuous phenotypic
polymorphism and its maintenance by natural selection are clear from
the writings of Ford cited above but these principles also can be
found together in a single statement: “In view of these
considerations it is clear that if any unifactorial character is at
all widespread it must be of some [adaptive] value. Indeed, it is
probably true that even if it occurs at as low a frequency as 1 per
cent, it must have been favored by selection” (Ford 1975,
p. 110). Thus, the primary goal of the ecological geneticist is to
discern exactly how natural selection is acting to maintain a
<em>balanced polymorphism</em> by the relative strength of opposing
fitness effects acting on the different sexes or at different stages
in the life history of the organism.</p>

<p>

The existence of males and females was discussed by Ford as a prime
example of a balanced polymorphism because, “It is obvious that
any tendency for the males to increase at the expense of the females,
or the reverse, would be opposed by selection” (Ford 1975, p.
111). Fisher (1930) first argued that, because every individual has a
mother and a father, the mean fitness of males must be equal to the
mean fitness of females multiplied by the sex ratio, expressed as the
number of females to males (i.e., the mean number of mates per male;
see also Shuster and Wade 2003, Chapter 1). As a result fitness
increases with rarity, and, in this circumstance, whenever the
population sex ratio deviates from unity, a gene that increases the
numbers of the minority sex at birth will have a selective advantage.
Thus, a sex ratio of unity is a stable, balanced polymorphism, achieved
in many species by chromosomal determination of sex, which Ford
referred to as a “‘built-in’ genetic
switch-mechanism”, characteristic of other genetic polymorphisms,
like Batesian mimicry. In general, the fitnesses of the different types
constituting a phenotypic polymorphism must be equal to be maintained
within a population by natural selection at a non-zero equilibrium
frequency (a point recognized by Darwin 1874, p. 275). However, the
balance of selective forces for non-sex related (or even sex-linked)
polymorphisms is very different from that required to maintain an equal
sex ratio, namely, the necessity that each offspring inherit equally
from each sex parent. Using the existence of the separate sexes as an
example of a balanced polymorphism is misleading or, at least
unrepresentative, of the selective forces necessary to sustain balanced
polymorphisms in general.</p>

<h2><a name="3">3. Classical Ecological Genetics, Population Size, and Natural Selection</a></h2>

<p>

The founding ecological geneticists dismissed any significant role
for random genetic drift in evolution. The theoretical interaction of
random genetic drift and natural selection for single genes with
constant effects can be seen in Figure 1. Fisher in his evolutionary
theory assumed that natural populations achieved or sustained the very
large sizes as seen in his in correspondence with S. Wright (cited in
Provine 1971) where he stated that “I believe N must usually be
the total population on the planet, enumerated at sexual
maturity”. Similarly, according to his intellectual
biographer W. Ewens (2000, p. 33): “Fisher never paid much
attention to the concept [effective population size] as he should have
… and used extremely high population sizes (up to
10<sup>12</sup>) in his analyses, surely far too large in
general.” For such extremely large population sizes, the
threshold between selection and drift (see Fig. 1), which is determined
by the effective population size,
<em>N<sub>e</sub></em>, is much lower. As a
result, the strength of random genetic drift, which is proportional to
(1/2<em>N<sub>e</sub></em>), is very, very weak and
even genes with very small values of <em>s</em> have
their evolutionary fate determined entirely by selection. This is the
essence of “<em>Fisherian gradualism</em>” — very
small selective forces given sufficient time can have effects on
adaptation similar to those of genes with much larger effects acting
over a shorter time period. With very large
<em>N<sub>e</sub></em>, the domain of random genetic
drift is greatly restricted even as that of natural selection is
expanded (see Fig. 1).</p>

<blockquote><img src="figure1.gif" alt="figure1" />
<br />
 <strong>Figure 1.</strong> The interaction of Random Genetic Drift and
Natural Selection. The strength of selection is measured by the
selective effect, <em>s</em>, of a single gene and the
strength of random genetic drift is indicated by
(1/2<em>N<sub>e</sub></em>), where the population has
effective size, <em>N<sub>e</sub></em>. When
<em>s</em> exceeds
(1/2<em>N<sub>e</sub></em>), then the evolutionary
fate of a gene is determined primarily by Natural Selection. When
<em>s</em> is less than
(1/2<em>N<sub>e</sub></em>), then the evolutionary
fate of a gene is determined primarily by Random Genetic Drift. Thus,
the evolutionary domains of natural selection (upper) and random
genetic drift (lower) are separated by the wavy boundary determined by
the effective population size.
</blockquote>

<p>

However, ecological geneticists did <em>not</em> dismiss random
genetic drift as a significant evolutionary force for the same reasons
that Fisher did. Field observations conducted with the
mark-recapture methods developed by ecological geneticists documented
generation-to-generation fluctuations in population size up to or
exceeding an order of magnitude in most natural populations studied
long term. Thus, small local population sizes were not seen as unusual
by ecological geneticists. Indeed, Ford believed that “…
organisms automatically generate their own cycles of abundance and
rarity and that the changes in selection pressure with which these are
associated many greatly increase the speed of evolution” (Ford
1975, p. 36). Despite the not infrequent occurrence of small
population sizes where drift would be expected to be most efficacious,
random genetic drift was considered an irrelevant evolutionary force in
ecological genetics because natural selection was viewed as being
particularly strong during periods of population decline. The smallest
populations showed little phenotypic variation, which was seen as
evidence that they were the most fit or most finely adapted
populations. The stressful environmental conditions responsible for the
decline in numbers also were seen as causing particularly strong
natural selection. Thus, the lack of phenotypic variation in small
populations was owing to it having been eliminated by natural selection
during the immediately prior period of decline. Conversely, under
periods of population increase, natural selection was seen as weaker
and more permissive of variation. This concept of relaxed selection
provided Ford with a cause for the increase in observations of rare
phenotypic variants in large and growing natural populations. If
selection pressure increases inversely to population size, then the
role of random genetic drift in evolution must be greatly
restricted.</p>

<p>

In addition, Ford (1975, p. 38) considered that ecological genetic
research had clearly demonstrated that the selective advantage of a
gene in nature ‘… quite commonly exceeds 25 per cent and
is frequently far more …” Referring to Figure 1, this
means that the range for values of <em>s</em> in
natural populations lies significantly above 0.01, placing genes in
very small populations firmly in the domain governed by natural
selection.</p>

<p>

Furthermore, Ford considered that not only the strength but also the
nature of selective pressures must frequently change with density
because “… an organism has not the same adaptive
requirements when abundant as when rare, or when the plant and animal
forms which impinge on it are so” (Ford 1975, p. 39). Indeed, he
thought that the fluctuating selection pressure caused by variations in
abundance ‘invalidates’ Wright's Shifting Balance Theory of
Evolution, which he referred to as ‘far-fetched’.
Interestingly, Ford and his colleagues believed that genetic
subdivision of the sort postulated by Wright would promote rapid
evolution but for very different genetic reasons and by different
genetic mechanisms (natural selection instead of random genetic drift,
local selection, and interdemic selection). Ford (1975, p. 40-44)
argued that subdivision of a large, geographically extensive population
into relatively small groups promotes rapid evolution because,
“… when populations occupy a series of restricted habitats
they can adapt themselves independently to the local environment in
each of them, while when spread over a larger area they can be adjusted
[by natural selection] only to the average of the diverse conditions
which obtain there. This, however, requires that the adaptations should
not be constantly broken down by a trickle of immigrants from one small
colony to another”. Here, he proposes a trade-off between
specialized adaptation to local conditions in the absence of migration
and generalized adaptation to global conditions in the presence of
migration. In modern terms, this is called genotype-by-environment
interaction, where the selective effect, <em>s</em>, of a gene changes
with change in the environment. A gene might be adaptive in one
environmental context (i.e., <em>s</em> &gt; 0) but maladaptive in
another (i.e., <em>s</em> &lt; 0). Migration between local environments
mixes the adaptive and maladaptive responses to selection and reduces
the average magnitude of gene frequency change. In this sense,
genotype-by-environment interaction is viewed as an evolutionary
constraint because is limits the rate of gene frequency change. 
The restraint can be removed simply by stopping gene flow or the mixing
of genes across different local environments. Thus, the fixed selective
effect illustrated in Figure 1, must be considered an <em>average</em>
selective effect across environments. Clearly, large local effects of
opposite sign must be averaged when there is gene flow among habitats
and the averaging tends to reduce the gene's selective effect. Ford
also suggests that the genetic mechanism involves “gene
complex[es] balanced to fit their own local environment”. That
is, he claims interactions among genes, or epistasis, contribute to
local adaptation. Thus, Ford invokes genotype-by-environment
interactions for fitness as well as gene-gene interactions for fitness
in his cases of rapid evolution. Both of these kinds of interactions
change the depiction of the threshold separating natural selection from
random genetic drift (Figure 1) in important ways (see below). Before
turning to interaction effects, I will examine a representative
discussion of ecological genetics of random genetic drift using data
from a natural population.</p>

<h2><a name="4">4. The Sewall Wright Effect</a></h2>

<p>

Several wing coloration variants segregating in a small natural
population of the moth, <em>Panaxia dominula</em> (Fisher and Ford
1947), were investigated using mark-recapture in one of the longest
continuous studies of a single population in evolutionary research. The
goal of Fisher and Ford was to determine whether year-to-year
fluctuations in the frequency of the variants (<em>medionigra</em>, a
heterozygote, and <em>bimaculata</em>, a homozygote) were better
explained by natural selection or by random genetic drift. They
inferred from their analysis</p>

<blockquote>“The conclusion that natural populations in general,
like that to which this study is devoted, are affected by selective
action, varying from time to time in direction and intensity and of
sufficient magnitude to cause fluctuating variations in all gene
frequencies is in good accordance with other studies of observable
frequencies in wild populations. We do not think, however, that it has
been sufficiently emphasized that this fact is fatal to the theory
which ascribes particular evolutionary importance to such fluctuations
in gene ratios as may occur by chance in very small isolated
populations… Thus our analysis, the first in which the relative
parts played by random survival and selection in a wild population can
be tested, does not support the view that chance fluctuations can be of
any significance in evolution.”</blockquote>

<p>

With this paper, Fisher and Ford moved the long-standing debate
between Wright and Fisher over the relative roles of natural selection
and random genetic drift in evolution from theory to nature. It is
remarkable that, in the first such study with only eight years of
observations on a single locus with alternative alleles, they are
confident in rejecting Wright's theory and random genetic drift in its
entirety. In his response (Wright 1948), Wright pointed out, first,
that his theory of evolution explicitly involved the simultaneous
action of several forces (selection, drift, mutation, and migration)
and he emphatically rejected the paradigm of Fisher and Ford that
<em>either</em> selection or drift alone had to be responsible for all
of the observed fluctuation in gene frequencies. Wright noted that, in
order to reach their statistical conclusion, Ford and Fisher had to
include gene frequency data from a decade before the more careful
study, notably a period without any estimates of population size.
Without this earlier data point, the average fluctuations were much
smaller and not significant. He pointed out that, like the
mark-recapture estimates of population numbers, the gene frequencies
themselves were estimates whose variation, based on the reported
sample sizes, accounted for more than half (55.2%) of the observed
variance that Fisher and Ford were trying to explain. He then showed
that, if one assumed only the unitary explanation of natural
selection, then the observed gene frequency fluctuations were so large
<em>even without the sampling variance</em> that the temporal
variations in the allelic selection coefficients must range from near
lethality (or sterility) to tremendous advantage (i.e., -0.50 to
+0.50). However, Fisher and Ford (1947) provided no indication of
comparable levels of temporal variation in any environmental factor
acting as a selective agent.  Wright argued that the effective
population sizes used in the analysis were almost certainly too large,
possibly by an order of magnitude, and that Fisher and Ford had made
no attempt to estimate the factors expected to reduce effective size,
like temporal variation in breeding numbers, non-random mortality
among larvae (mortality clustered within families as might affect a
species which experiences &gt; 85% pupal mortality owing to viral
infection), or other causes of the variance in offspring numbers (such
as variation among females in egg numbers or variation among males in
mate numbers). In an unyielding reply, Fisher and Ford (1950) labeled
chance or random fluctuations in gene frequency, the <em>Sewall Wright
Effect</em>, a term which has endured to the present day as a synonym
for random genetic drift.</p>

<p>

With a larger data set covering several more years, Ford (1975, p.
146) revisited this exchange and argued that Wright remained wrong on
each count. Ford also showed that the selective advantage for the rarer
of the genes varied widely, from -0.10 to +0.20, and that there was no
evidence of heterozygote advantage. He did not find, however, the
expected negative correlation between strength of selection and
population size in these data. In the intervening decades, data from a
variety of other organisms and natural population had become available
and its review led Ford (1975, p. 389) to conclude: “As a result,
it is no longer possible to attribute to random genetic drift or to
mutation any significant part in the control of evolution.” Thus,
throughout its founding period, ecological genetics was relentlessly
supportive of natural selection as unitary explanation for evolutionary
change. (Later laboratory research has shown that the expression of the
color patterns is sensitive to the thermal environment during
development and thus the gene frequency estimates may be subject to
significant measurement error, owing to the misclassification of
genotypes. This is yet another source of variation, not accounted for
in the Ford analyses. In addition, empirical evidence has found, as
Wright expected, that temporal fluctuations in population size, large
variance among females in fecundity, and sexual selection reduce the
effective number to less than half the Fisher-Ford estimate. In
addition, more careful studies have reduced Ford's estimates of the
magnitude of the average genic selection coefficient by about two
thirds [cf. Cook and Jones 1996].)</p>

<h2><a name="5">5. Interactions and their Effect on the Threshold between Natural Selection and Random Drift</a></h2>

<p>

The existence of either genotype-by-environment interaction (G ×
E) or gene-by-gene interaction (epistasis or G × G) greatly
complicates the estimation of selection coefficients.  Ecological
geneticists like Ford postulated interactions of the sort that could
change the sign of genic selection coefficients with changes in the
environment (including density) or in the genetic background.  This
kind of reversal of selective effect requires what is known as a
‘crossing-type’ norm of reaction for G × E or
additive-by-additive epistasis for G × G (Wade 2002). The
simplest model of crossing-type G × E, consists of additive
selection (i.e., genotypic fitnesses of 1 + 2<em>s</em>, 1 +
<em>s</em>, and 1 for genotypes <em>AA</em>, <em>Aa</em>, and
<em>aa</em>, in one environment and the opposite order in the second
environment) in each of two alternative environments, E<sub>1</sub>
and E<sub>2</sub>, with frequencies, f<sub>E1</sub> and
f<sub>E2</sub>, respectively. As the two environments fluctuate in
frequency, spatially or temporally, the selective effect of an
<em>A</em> allele changes in both magnitude and sign (see Figure
2). Depending upon the relative frequencies of the alternative
environments and the amount of gene flow or migration between them,
the <em>A</em> allele on average can be a ‘good’ gene or a
‘bad’ gene, a gene of major effect or minor effect, or
even a neutral gene if the two environments are equally abundant. The
smaller the amount of migration between the environments, the greater
is the degree of local adaptation to each as Ford suggested (see
above). However, the <em>average</em> selective effect of the gene in
the sense of Fisher's theory must be smaller than the average
observation in a particular locality at a particular time because the
long-term average contains both positive and negative values of
<em>s</em>. Furthermore, to the extent that the local value of
<em>s</em> changes sign owing to continuous fluctuations in local
environmental conditions, the <em>A</em> allele will also move from
the domain of selection to the domain of drift as Wright
suggested. Thus, the very kind of population subdivision imagined by
Ford, with selection acting in every locality albeit in different
directions, creates, rather than eliminates, the opportunity for
random genetic drift.  </p>

<blockquote><img src="figure2.gif" alt="figure2" />
<br />
 <strong>Figure 2.</strong> The interaction of Random Genetic Drift
and Natural Selection, when there is either genotype-by-environment
interaction or additive-by-additive epistasis (see text). The
selective effect, <em>s</em>, of a single gene changes magnitude as
the frequency of the alternative environments, f<sub>E1</sub> and
f<sub>E2</sub>, connected by gene flow, changes <em>or</em> as the
frequency of alternative alleles, <em>p</em><sub><em>B</em></sub> and
<em>p</em><sub><em>b</em></sub>, change at an interacting locus. Thus,
neither the selective effect of a gene nor the effective population
size remains constant. As a result, relative to Figure 1, the
threshold boundary between the domains of Natural Selection and Random
Genetic Drift is greatly widened, meaning that both forces play more
or less equal evolutionary roles over a broad range of values of
<em>s</em> and <em>N<sub>e</sub></em>. Furthermore, interactions of
this sort open the possibility that changes in the relative
frequencies of alternative environments or alternative alleles at
other loci can move a gene's selective effect from the domain of
selection that of drift or vice versa during the course of its
evolution.  </blockquote>

<p>

A very similar effect on the ‘gene's eye view’ of
selection is caused by additive-by-additive epistasis (Goodnight and
Wade 2000; Wade 2001, 2002). The simplest model of this kind of G
× G, with interaction between loci <em>A</em> and <em>B</em>,
each with alternative alleles, results in an average genic selection
coefficient acting on the <em>A</em> allele of
<em>s</em>(<em>p</em><sub><em>B</em></sub> −
<em>p</em><sub><em>b</em></sub>). The relative frequencies of the
alternative alleles at the <em>B</em> locus, determine whether the
<em>A</em> allele is a ‘good’ gene or a ‘bad’
gene, a gene of major effect or minor effect, or even a neutral gene
when the alleles are equally abundant (i.e.,
<em>p</em><sub><em>B</em></sub> =
<em>p</em><sub><em>b</em></sub>). Whenever allele frequencies of its
epistatic partner change, either by drift or selection, the <em>A</em>
allele's selective effect also changes and, like the case of G ×
E, it moves between the domains of natural selection and random drift
(Figure 2).</p>

<h2><a name="6">6. Allozyme Variation and the Drift vs Selection Controversy</a></h2>

<p>

The central problem with using conspicuous polymorphisms for
investigating the relative roles of the variety of different
evolutionary forces is that it is not an unbiased sample of genetic
diversity with respect to either degree of adaptive function or amount
of genetic variation. Indeed, the definition of genetic polymorphism
adopted by Ford (see above) incorporates the essence of both of these
biases. For a period, it was believed that “The solution to our
dilemma lies in the development of molecular genetics” (Lewontin
1974, p. 99). With the advent of electrophoresis, the amino acid
sequence of a random sample of proteins from almost any organism could
be studied and, for the first time, the level of genetic diversity, in
the form of amino acid substitutions, across the genome could be
quantified.</p>

<p>

Two measures of genetic diversity were possible using
electrophoresis: (1) the number of loci polymorphic; and, (2) the
average heterozygosity per an individual. From studies across a number
of species, it was estimated that 15-40% of all loci were polymorphic
and the average individual was heterozygous at 5-15% of its genome.
Since this technique measured primarily amino acid substitutions
resulting in charge changes, i.e., only one third of all possible amino
acid substitutions, one could infer that these were minimal levels of
genetic diversity. The conclusion that genetic variation was
ubiquitous, with most genes being polymorphic, was inescapable. The
search for the adaptive function of allozyme variants and balancing
selection at the physiological level ensued.</p>

<p>

However, these levels of genetic polymorphism appeared to be much too
large to be explained by the type of balancing selection observed by
Ford and his colleagues for conspicuous phenotypic polymorphisms in
natural populations. The basic problem was that the numbers of
selective deaths necessary to account for the observed levels of
allozyme polymorphism exceeded the reproductive excess of almost all
species. Haldane (1957) called this the “cost of natural
selection” and it is also referred to as the substitutional
load.  Differently put, the mortality of homozygous genotypes, if
independently selected, (also known as the ‘segregation
load’) would exceed the total numbers of offspring produced by a
popualtion. For this reason, Kimura (1983) proposed his neutral theory
of molecular evolution, founded on the theoretical observation that
the probability of fixation of a novel mutant allele with selective
coefficient, <em>s</em> &gt;0, was approximately 2<em>s</em>. Thus,
the probability of loss of even a favored mutation was, for small
<em>s</em>, only slightly smaller than the probability of loss by
chance for a truly neutral allele. Studies of protein structure also
revealed that the functional sites of a protein, which constitute the
minority of its amino acids, evolved several times more slowly than
the non-functional or structural sites. The view that much, if not
most, of evolutionary change at the molecular level was determined by
random genetic drift and not natural selection was highly
controversial. As Kimura noted (1983, p. 22), “…if a
certain doctrine is constantly being spoken of favorably by the
majority, endorsed by top authorities in their books and taught in
classes, then a belief is gradually built up in one's mind, eventually
becoming the guiding principle and the basis of value judgment. At any
rate, this was the time when the panselectionist or
‘neo-Darwinian’ position was most secure in the history of
biology: the heyday of the traditional ‘synthetic theory’
of evolution.”</p>

<p>

It was soon recognized that a more reductionistic approach (DNA
sequence studies) might help to resolve the issue of whether or not
every amino acid was of some functional value because the redundant
positions in the code of life were assumed to provide an estimate of
the true ‘neutral’ rate of evolution, owing to random
genetic drift acting in the absence of selection.</p>

<h2><a name="7">7. Sequence Variation and the Drift vs Selection Controversy</a></h2>

<p>

The neutral theory of evolution is the antithesis of ecological
genetics. It states that random genetic drift, rather than natural
selection, governs most evolutionary change at the level of the DNA and
proteins, while admitting that natural selection predominates in
shaping the morphological and physiological traits that manifest an
adaptive fit with the environment. This is a paradox because most of
the DNA appears to be non-functional while most of the externally
observable phenotype appears to have adaptive function.</p>

<p>

Tests of the theory using DNA sequence data consist of comparisons
of the relative evolutionary rates of different kinds of sites (base
pairs) within codons and take advantage of the redundancy in the
genetic code. The rate of neutral evolution is estimated from levels of
polymorphism or numbers of segregating sites within species or the
divergence between species in silent or redundant site substitutions.
Silent sites are those that do not result in an amino acid change in
the protein and, hence, are non-functional in the usual sense. In
contrast, the rate of selective change or selective constraint is
evaluated relative to the neutral rate using replacement sites, those
base pair changes that result in amino acid changes. If the rate of
substitution or polymorphism is lower than neutral, it is evidence of
selective constraint or purifying natural selection acting to prevent
change and preserve function in the face of mutational damage. If the
rate of substitution is higher than neutral, then it is evidence of
adaptive substitution. </p>

<p>

Molecular evolutionary studies also revealed the existence of
<em>pseudogenes</em>, non-coding stretches of DNA derived by the
tandem duplication and subsequent inactivation by mutation of single
copy genes. The lack of function of the pseudogene makes all of its
codons effectively neutral and provides another estimate of the rate
of neutral evolution. Importantly, ‘replacement’ sites
that evolve slowly in the functional gene have been shown to evolve
more rapidly in the nonfunctional tandem duplicate pseudogene.</p>

<p>

Changes in the pattern of neutral variation in the vicinity of a
selected site(s) are also informative because, during an adaptive
substitution, neutral variants linked to the piece of selected DNA are
carried or ‘swept’ to fixation along with it. This
‘selective sweep’ temporarily reduces the level of neutral
variation in the vicinity of selected sites until it can be replaced by
mutation. The degree of reduction in neutral variation or the
‘footprint of selection’ depends upon the strength of
selection, the frequency of recombination during selection, and the
time since the initiation of selection. The footprint is most
conspicuous when a selective sweep is initiated by the advent of a
single, novel favorable mutation. To the extent that novel selection
results from a change of environment and begins to act on existing or
standing variation already in the population, the impact on neutral
polymorphisms may be quite minimal. Balancing selection of the sort
observed by Ford leaves its own unique ‘reverse’ footprint
on neutral diversity. Because the segments of DNA constituting
the balanced polymorphism are held in the population by selection much
longer than expected based on random drift, these segments have a
higher effective population size (owing to lower variation in offspring
numbers than random) and tend to accumulate mutational variation at
nearby neutral sites. Thus, levels of neutral diversity are expected to
be enhanced in the vicinity of a molecular balanced polymorphism. When
the mating system restricts recombination (e.g., in selfing or
inbreeding species), the region of elevated neutral diversity in the
vicinity of a balanced polymorphism can be extensive.</p>

<p>

Kimura predicted that silent substitutions would evolve more rapidly
than replacement substitutions <em>before</em> sequence data were
available to test his neutral theory of molecular evolution. Molecular
genetic studies have confirmed his prediction: silent sites evolve
several times faster than replacement sites. These studies clearly show
that the primary mode of action of natural selection at the level of
the DNA sequence is purifying selection. It is this highly conservative
aspect of natural selection that permits comparative molecular
evolutionary studies of developmental processes across species as
diverse as humans and flies. At the molecular level, most genes, though
polymorphic in sequence, do not display evidence of balancing selection
and instead manifest patterns of variation that accord well with
neutral theory.</p>

<p>

The interaction of selection and random drift across linked regions
of DNA sequence is one of the most active current areas of theoretical
and empirical research in molecular evolution. Theory shows that it can
be difficult to separate cleanly the action of the evolutionary forces
of selection and drift except for certain regions of parameter space,
whose generality remains unknown and subject to much debate. Like the
study by Fisher and Ford (1947), most empirical studies interpret
<em>all</em> deviations away from strictly the neutral expectation as
evidence of natural selection without addressing the issue of agency.
Thus, nonrandom or biased use of redundant codons in some regions of
DNA sequence has been documented. Codon bias is seen as evidence that,
although they have no effect on amino acid sequence, redundant codons
are not all functionally equivalent. This is taken as evidence that
natural selection is all powerful, reaching down into the genome to
affect even the smallest and least significant components of the
hereditary material. Thus, the original ecological genetic view that
natural selection is the only significant evolutionary force
characterizes much of molecular evolution, despite progress in theory
and the availability of much more reductionistic genetic methods. The
parallels between the summary statement of Ford (1975, p. 389; see
above) and that of the molecular evolutionary geneticist, E. Nevo
(2001, p. 6223), twenty-five years later are remarkable:
“Biodiversity evolution, even in small isolated populations, is
primarily driven by natural selection, including diversifying,
balancing, cyclical, and purifying selective regimes, interacting with,
but ultimately overriding, the effects of mutation, migration, and
stochasticity.”</p>

</div>

<div id="bibliography">

<h2><a name="Bib">Bibliography</a></h2>

<ul>
<li>Cook, L. M. and D. A. Jones. 1996. "The <em>medionigra</em> gene in
the moth <em>Panaxia dominula</em>: The case for selection." <em>Phil.
Trans. Roy. Soc.</em>, B 351: 1623-1634.</li>

<li>Ewens, W. C. 2000. "The mathematical foundations of population
genetics." In: <em>Evolutionary genetics</em>, R. Singh and C.
Krimbas, eds. Cambridge University Press, New York,  pp. 24-40.</li>

<li>Fisher, R. A. 1930. <em>The genetical theory of natural
selection</em>.  Clarendon Press, Oxford.</li>

<li>Fisher, R. A., and E. B. Ford. 1947. "The spread of a gene in
natural conditions in a colony of the moth <em>Panaxia dominula</em>
L."  <em>Heredity</em>, 1:143-174.</li>

<li>Fisher, R. A., and E. B. Ford. 1950. "The “Sewall
Wright” effect." <em>Heredity</em> 4:117-119.</li>

<li>Ford, E. B. 1940. "Genetic research in the Lepidoptera." <em>Ann.
Eugenics</em>, 10:227-252.</li>

<li>Ford, E. B. 1975. <em>Ecological Genetics</em>, 4th ed,
Chap. 7. Chapman and Hall, London.</li>

<li>Goodnight, C. J. and M. J. Wade. 2000. "The ongoing synthesis: a
reply to Coyne <em>et al</em>." (1999). <em>Evolution</em>, 54:
317-324.</li>

<li>Haldane, J. B. S. 1957. "The cost of natural selection."
<em>Journal of Genetics</em>, 55:511-524.</li>

<li>Huxley, J. 1942. <em>Evolution the modern synthesis</em>. Harper
&amp; Brothers, New York.</li>

<li>Kimura, M. 1983. <em>The neutral theory of molecular
evolution</em>.  Cambridge: Cambridge University Press.</li>

<li>Lewontin, R. C. 1974. <em>The genetic basis of evolutionary
change</em>.  New York: Columbia University Press.</li>

<li>Nevo, E. 2001. "Evolution of genome-phenome diversity under
environmental stress." <em>Proc. Natl. Acad. Sci.</em> (USA),
98:6233-6240.</li>

<li>Provine, W. B. 1971. <em>The origins of theoretical population
genetics</em>. Chicago: University of Chicago Press.</li>

<li>Provine, W. B. 1986. <em>Sewall Wright and evolutionary
biology</em>.  Chicago: University of Chicago Press.</li>

<li>Shuster, S. M. and M. J. Wade. 2003. <em>Mating systems and
strategies</em>.  Princeton: Princeton University Press.</li>

<li>Wade, M. J. 2001. "Epistasis, complex traits, and rates of
evolution." <em>Genetica</em>, 112: 59-69. </li>

<li>Wade, M. J. 2002. "A gene's eye view of epistasis, selection, and
speciation." <em>J. Evol. Biology</em>, 15:337-346.</li>

<li>Wade, M. J., and C. J. Goodnight. 1998. "Genetics and adaptation
in metapopulations: When nature does many small experiments."
<em>Evolution</em>,  52:1537-1553.</li>

<li>Wright, S. 1948. "On the roles of directed and random changes in
gene frequency in the genetics of populations." <em>Evolution</em>,
2:279-294.</li>

 </ul>

</div>

<div id="academic-tools">

<h2><a id="Aca">Academic Tools</a></h2>

<blockquote>
<table>
<tbody><tr><td valign="top"><img src="../../symbols/sepman-icon.jpg" alt="sep man icon" /></td>
<td><a href="https://plato.stanford.edu/cgi-bin/encyclopedia/archinfo.cgi?entry=evolutionary-genetics&amp;archive=win2019" target="other">How to cite this entry</a>.</td>
</tr>
<tr><td valign="top"><img src="../../symbols/sepman-icon.jpg" alt="sep man icon" /></td>
<td><a href="https://leibniz.stanford.edu/friends/preview/evolutionary-genetics/" target="other">Preview the PDF version of this entry</a> at the <a href="https://leibniz.stanford.edu/friends/" target="other">Friends of the SEP Society</a>.</td>
</tr>
<tr><td valign="top"><img src="../../symbols/inpho.png" alt="inpho icon" /></td>
<td><a href="https://www.inphoproject.org/entity?sep=evolutionary-genetics&amp;redirect=True" target="other">Look up this entry topic</a> at the <a href="https://www.inphoproject.org/" target="other">Indiana Philosophy Ontology Project</a> (InPhO).</td>
</tr>
<tr><td valign="top"><img src="../../symbols/pp.gif" alt="phil papers icon" /></td>
<td><a href="http://philpapers.org/sep/evolutionary-genetics/" target="other">Enhanced bibliography for this entry</a> at <a href="http://philpapers.org" target="other">PhilPapers</a>, with links to its database.</td>
</tr>
</tbody></table>
</blockquote>

</div>

<div id="other-internet-resources">

<h2><a name="Oth">Other Internet Resources</a></h2>

<p>

[Please contact the author with suggestions.]</p>

</div>

<div id="related-entries">

<h2><a name="Rel">Related Entries</a></h2>

<p>

 <a href="../molecular-genetics/">genetics: molecular</a> |
 <a href="../population-genetics/">genetics: population</a> |
 <a href="../natural-selection/">natural selection</a>

</p>

</div>

</div><!-- #aueditable --><!--DO NOT MODIFY THIS LINE AND BELOW-->

<!-- END ARTICLE HTML -->

</div> <!-- End article-content -->

  <div id="article-copyright">
    <p>
 <a href="../../info.html#c">Copyright © 2005</a> by

<br />
Michael Wade

    </p>
  </div>

</div> <!-- End article -->

<!-- NOTE: article banner is outside of the id="article" div. -->
<div id="article-banner" class="scroll-block">
     <div id="article-banner-content">
  <a href="../../fundraising/">
  Open access to the SEP is made possible by a world-wide funding initiative.<br />
  Please Read How You Can Help Keep the Encyclopedia Free</a>
 </div>


</div> <!-- End article-banner -->

    </div> <!-- End content -->

    <div id="footer">

      <div id="footer-menu">
        <div class="menu-block">
          <h4><i class="icon-book"></i> Browse</h4>
          <ul role="menu">
            <li><a href="../../contents.html">Table of Contents</a></li>
            <li><a href="../../new.html">New in this Archive</a></li>
            
            <li><a href="../../published.html">Chronological</a></li>
            <li><a href="../../../../archives/">Archives <i class="icon-external-link"></i></a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-info-sign"></i> About</h4>
          <ul role="menu">
            <li><a href="../../info.html">Editorial Information</a></li>
            <li><a href="../../about.html">About the SEP</a></li>
            <li><a href="../../board.html">Editorial Board</a></li>
            <li><a href="../../cite.html">How to Cite the SEP</a></li>
            <li><a href="../../special-characters.html">Special Characters</a></li>
            
            <li><a href="../../../../contact.html">Contact <i class="icon-external-link"></i></a></li>
          </ul>
        </div>
        <div class="menu-block">
          <h4><i class="icon-leaf"></i> Support SEP</h4>
          <ul role="menu">
            <li><a href="../../../../support/">Support the SEP</a></li>
            <li><a href="../../../../support/friends.html">PDFs for SEP Friends</a></li>
            <li><a href="../../../../support/donate.html">Make a Donation</a></li>
            <li><a href="../../../../support/sepia.html">SEPIA for Libraries</a></li>
          </ul>
        </div>
      </div> <!-- End footer menu -->

      <div id="mirrors">
        <div id="mirror-info">
          <h4><i class="icon-globe"></i> Mirror Sites</h4>
          <p>View this site from another server:</p>
        </div>
                <div class="btn-group">
<a class="btn dropdown-toggle" data-toggle="dropdown" href="https://plato.stanford.edu/"><span class="flag flag-usa"></span> USA (Main Site) <span class="caret"></span><span class="mirror-source">CSLI, Stanford University</span></a>          <ul class="dropdown-menu">
            <li><a href="https://stanford.library.sydney.edu.au/archives/win2019/entries/evolutionary-genetics/"><span class="flag flag-australia"></span> Australia <span class="mirror-source">Library, University of Sydney</span></a>           </li>
            <li><a href="https://seop.illc.uva.nl/archives/win2019/entries/evolutionary-genetics/"><span class="flag flag-netherlands"></span> Netherlands <span class="mirror-source">ILLC, University of Amsterdam</span></a>           </li>
          </ul>
        </div>
      </div> <!-- End mirrors -->
      
      <div id="site-credits">
        <p class="csli-logo"><a href="https://www-csli.stanford.edu/"><img src="../../symbols/SU_csli.png" width="355" alt="Stanford Center for the Study of Language and Information" /></a></p>
        <p>The Stanford Encyclopedia of Philosophy is <a href="../../info.html#c">copyright © 2016</a> by <a href="http://mally.stanford.edu/">The Metaphysics Research Lab</a>, Center for the Study of Language and Information (CSLI), Stanford University</p>
        <p>Library of Congress Catalog Data: ISSN 1095-5054</p>
      </div> <!-- End site credits -->

    </div> <!-- End footer -->

  </div> <!-- End container -->

   <!-- NOTE: Script required for drop-down button to work (mirrors). -->
  <script>
    $('.dropdown-toggle').dropdown();
  </script>





</body></html>